generator/generator.go

```diff
@@ -206,6 +206,18 @@ func generateInitRamfs(conf *generatorConfig) error {
 		}
 	}
 
+	// force load usbhid and hid_sensor_hub modules when unlocking encrypted LUKS2 volumes with fido2 tokens/devices
+	if conf.extraFiles != nil {
+		for _, file := range conf.extraFiles {
+			if file == "fido2-assert" {
+				conf.modulesForceLoad = append(conf.modulesForceLoad, "usbhid")
```

Comment:
you can merge these 2 lines into 1:

```go
conf.modulesForceLoad = append(conf.modulesForceLoad, "usbhid", "hid_sensor_hub")
```

Reply:
I actually have a followup question - what is the reason of using `modulesForceLoad` instead of a more relaxed `modules` option?

I expect that "usbhid", "hid_sensor_hub" modules loading should be handled by udev.

Reply:
It's to ensure those modules are loaded before booster asks for the FIDO2 key. Both the generated and minimal image already include them by default, so I didn't see the point of using the `modules` option. Without it forcibly loaded, when booster looks for hidraw devices, it is unable to read the uevent. But for some reason, it's observed a universal image **always** loads them before the key is asked.

Here are my logs to support this:

booster.yaml
```
modules_force_load: amdgpu,usbhid,hid_sensor_hub
extra_files: fido2-assert
```
<details>
<summary>Minimal</summary>
<pre>
Aug 03 10:01:23 archlinux booster: unable to read uevent file for hidraw1
Aug 03 10:01:23 archlinux booster: HID hidraw0 does not support FIDO
Aug 03 10:01:23 archlinux booster: loading module cbc
Aug 03 10:01:23 archlinux booster: loading module dm_mod
Aug 03 10:01:23 archlinux booster: loading module tee
Aug 03 10:01:23 archlinux booster: loading module asn1_encoder
Aug 03 10:01:23 archlinux booster: a mapping for LUKS device /dev/nvme0n1p2 has been found
Aug 03 10:01:23 archlinux booster: blkinfo for /dev/nvme0n1p2: type=luks UUID=5e975a1e-da7a-424a-91c9-e58d441f9677 LABEL=
Aug 03 10:01:23 archlinux booster: blkinfo for /dev/nvme0n1p1: type=fat UUID=dce4f928 LABEL=NO NAME
Aug 03 10:01:23 archlinux booster: found a new device /dev/nvme0n1p2
Aug 03 10:01:23 archlinux booster: found a new device /dev/nvme0n1p1
Aug 03 10:01:23 archlinux booster: blkinfo for /dev/nvme0n1: type=gpt UUID=5d41ee56-196a-4696-aefc-9663bca535e5 LABEL=
Aug 03 10:01:23 archlinux booster: udev event {Action:add KObj:/devices/pci0000:00/0000:00:02.4/0000:02:00.0/nvme/nvme0/ng0n1 Env:map[ACTION:add DEVNAME:ng0n1 DEVPATH:/dev>
Aug 03 10:01:23 archlinux booster: udev event {Action:add KObj:/devices/pci0000:00/0000:00:02.4/0000:02:00.0/nvme/nvme0/nvme0n1/nvme0n1p2 Env:map[ACTION:add DEVNAME:nvme0n>
Aug 03 10:01:23 archlinux booster: found a new device /dev/nvme0n1
Aug 03 10:01:23 archlinux booster: udev event {Action:add KObj:/devices/pci0000:00/0000:00:02.4/0000:02:00.0/nvme/nvme0/nvme0n1/nvme0n1p1 Env:map[ACTION:add DEVNAME:nvme0n>
Aug 03 10:01:23 archlinux booster: udev event {Action:add KObj:/devices/pci0000:00/0000:00:02.4/0000:02:00.0/nvme/nvme0/nvme0n1 Env:map[ACTION:add DEVNAME:nvme0n1 DEVPATH:>
Aug 03 10:01:23 archlinux booster: udev event {Action:add KObj:/devices/virtual/bdi/259:0 Env:map[ACTION:add DEVPATH:/devices/virtual/bdi/259:0 SEQNUM:1523 SUBSYSTEM:bdi]}
Aug 03 10:01:23 archlinux booster: udev event {Action:change KObj:/devices/pci0000:00/0000:00:02.4/0000:02:00.0/nvme/nvme0 Env:map[ACTION:change DEVNAME:nvme0 DEVPATH:/dev>
Aug 03 10:01:23 archlinux booster: udev event {Action:add KObj:/devices/pci0000:00/0000:00:02.4/0000:02:00.0/nvme/nvme0/hwmon3 Env:map[ACTION:add DEVPATH:/devices/pci0000:>
Aug 03 10:01:23 archlinux booster: udev event {Action:add KObj:/module/libps2 Env:map[ACTION:add DEVPATH:/module/libps2 SEQNUM:1520 SUBSYSTEM:module]}
Aug 03 10:01:23 archlinux booster: udev event {Action:add KObj:/module/vivaldi_fmap Env:map[ACTION:add DEVPATH:/module/vivaldi_fmap SEQNUM:1519 SUBSYSTEM:module]}
Aug 03 10:01:23 archlinux booster: udev event {Action:add KObj:/module/serio_raw Env:map[ACTION:add DEVPATH:/module/serio_raw SEQNUM:1518 SUBSYSTEM:module]}
Aug 03 10:01:23 archlinux booster: udev event {Action:add KObj:/bus/serio/drivers/serio_raw Env:map[ACTION:add DEVPATH:/bus/serio/drivers/serio_raw SEQNUM:1517 SUBSYSTEM:d>
Aug 03 10:01:23 archlinux booster: udev event {Action:add KObj:/devices/virtual/nvme-subsystem/nvme-subsys0 Env:map[ACTION:add DEVPATH:/devices/virtual/nvme-subsystem/nvme>
Aug 03 10:01:23 archlinux booster: udev event {Action:add KObj:/module/hid_multitouch Env:map[ACTION:add DEVPATH:/module/hid_multitouch SEQNUM:1515 SUBSYSTEM:module]}
Aug 03 10:01:23 archlinux booster: udev event {Action:bind KObj:/devices/platform/AMDI0010:03/i2c-0/i2c-ELAN074E:00/0018:04F3:31B5.0001 Env:map[ACTION:bind DEVPATH:/device>
Aug 03 10:01:23 archlinux booster: udev event {Action:add KObj:/devices/platform/AMDI0010:03/i2c-0/i2c-ELAN074E:00/0018:04F3:31B5.0001/hidraw/hidraw0 Env:map[ACTION:add DE>
Aug 03 10:01:23 archlinux booster: no match found for alias input:b0018v04F3p31B5e0100_e0,1,3,4,k110,145,148,14A,14D,14E,14F,ra0,1,2F,35,36,37,39,m5,lsfw
Aug 03 10:01:23 archlinux booster: udev event {Action:add KObj:/devices/platform/AMDI0010:03/i2c-0/i2c-ELAN074E:00/0018:04F3:31B5.0001/input/input8/event4 Env:map[ACTION:a>
Aug 03 10:01:23 archlinux booster: udev event {Action:add KObj:/devices/platform/AMDI0010:03/i2c-0/i2c-ELAN074E:00/0018:04F3:31B5.0001/input/input8 Env:map[ABS:2e080000000>
Aug 03 10:01:23 archlinux kernel: usb 3-3: new full-speed USB device number 2 using xhci_hcd
Aug 03 10:01:23 archlinux kernel: usb 1-1: new full-speed USB device number 2 using xhci_hcd
Aug 03 10:01:23 archlinux booster: udev event {Action:add KObj:/devices/platform/AMDI0010:03/i2c-0/i2c-ELAN074E:00/0018:04F3:31B5.0001/input/input6/event3 Env:map[ACTION:a>
Aug 03 10:01:23 archlinux booster: udev event {Action:add KObj:/devices/platform/AMDI0010:03/i2c-0/i2c-ELAN074E:00/0018:04F3:31B5.0001/input/input6 Env:map[ACTION:add DEVP>
Aug 03 10:01:23 archlinux booster: udev event {Action:add KObj:/module/amd_sfh Env:map[ACTION:add DEVPATH:/module/amd_sfh SEQNUM:1508 SUBSYSTEM:module]}
Aug 03 10:01:23 archlinux booster: udev event {Action:add KObj:/bus/pci/drivers/pcie_mp2_amd Env:map[ACTION:add DEVPATH:/bus/pci/drivers/pcie_mp2_amd SEQNUM:1507 SUBSYSTEM>
Aug 03 10:01:23 archlinux booster: udev event {Action:remove KObj:/devices/0020:1022:0001.0004 Env:map[ACTION:remove DEVPATH:/devices/0020:1022:0001.0004 HID_ID:0020:00001>
Aug 03 10:01:23 archlinux booster: udev event {Action:unbind KObj:/devices/0020:1022:0001.0004 Env:map[ACTION:unbind DEVPATH:/devices/0020:1022:0001.0004 HID_ID:0020:00001>
Aug 03 10:01:23 archlinux booster: udev event {Action:remove KObj:/devices/0020:1022:0001.0004/hidraw/hidraw3 Env:map[ACTION:remove DEVNAME:hidraw3 DEVPATH:/devices/0020:1>
Aug 03 10:01:23 archlinux booster: udev event {Action:remove KObj:/devices/0020:1022:0001.0003 Env:map[ACTION:remove DEVPATH:/devices/0020:1022:0001.0003 HID_ID:0020:00001>
Aug 03 10:01:23 archlinux booster: udev event {Action:unbind KObj:/devices/0020:1022:0001.0003 Env:map[ACTION:unbind DEVPATH:/devices/0020:1022:0001.0003 HID_ID:0020:00001>
Aug 03 10:01:23 archlinux booster: udev event {Action:remove KObj:/devices/0020:1022:0001.0003/hidraw/hidraw2 Env:map[ACTION:remove DEVNAME:hidraw2 DEVPATH:/devices/0020:1>
Aug 03 10:01:23 archlinux booster: udev event {Action:remove KObj:/devices/0020:1022:0001.0002 Env:map[ACTION:remove DEVPATH:/devices/0020:1022:0001.0002 HID_ID:0020:00001>
Aug 03 10:01:23 archlinux booster: udev event {Action:unbind KObj:/devices/0020:1022:0001.0002 Env:map[ACTION:unbind DEVPATH:/devices/0020:1022:0001.0002 HID_ID:0020:00001>
Aug 03 10:01:23 archlinux booster: udev event {Action:remove KObj:/devices/0020:1022:0001.0002/hidraw/hidraw1 Env:map[ACTION:remove DEVNAME:hidraw1 DEVPATH:/devices/0020:1>
Aug 03 10:01:23 archlinux booster: udev event {Action:unbind KObj:/devices/platform/AMDI0010:03/i2c-0/i2c-ELAN074E:00/0018:04F3:31B5.0001 Env:map[ACTION:unbind DEVPATH:/de>
Aug 03 10:01:23 archlinux booster: udev event {Action:remove KObj:/devices/platform/AMDI0010:03/i2c-0/i2c-ELAN074E:00/0018:04F3:31B5.0001/hidraw/hidraw0 Env:map[ACTION:rem>
Aug 03 10:01:23 archlinux booster: udev event {Action:remove KObj:/devices/platform/AMDI0010:03/i2c-0/i2c-ELAN074E:00/0018:04F3:31B5.0001/input/input5 Env:map[ABS:10000000>
Aug 03 10:01:23 archlinux booster: udev event {Action:bind KObj:/devices/0020:1022:0001.0004 Env:map[ACTION:bind DEVPATH:/devices/0020:1022:0001.0004 DRIVER:hid-generic HI>
Aug 03 10:01:23 archlinux booster: udev event {Action:add KObj:/devices/0020:1022:0001.0004/hidraw/hidraw3 Env:map[ACTION:add DEVNAME:hidraw3 DEVPATH:/devices/0020:1022:00>
Aug 03 10:01:23 archlinux booster: udev event {Action:add KObj:/devices/0020:1022:0001.0004 Env:map[ACTION:add DEVPATH:/devices/0020:1022:0001.0004 HID_ID:0020:00001022:00>
Aug 03 10:01:23 archlinux booster: udev event {Action:bind KObj:/devices/0020:1022:0001.0003 Env:map[ACTION:bind DEVPATH:/devices/0020:1022:0001.0003 DRIVER:hid-generic HI>
Aug 03 10:01:23 archlinux booster: udev event {Action:add KObj:/devices/0020:1022:0001.0003/hidraw/hidraw2 Env:map[ACTION:add DEVNAME:hidraw2 DEVPATH:/devices/0020:1022:00>
Aug 03 10:01:23 archlinux booster: udev event {Action:add KObj:/devices/0020:1022:0001.0003 Env:map[ACTION:add DEVPATH:/devices/0020:1022:0001.0003 HID_ID:0020:00001022:00>
Aug 03 10:01:23 archlinux booster: udev event {Action:bind KObj:/devices/0020:1022:0001.0002 Env:map[ACTION:bind DEVPATH:/devices/0020:1022:0001.0002 DRIVER:hid-generic HI>
Aug 03 10:01:23 archlinux booster: no match found for alias hid:b0020g0003v00001022p00000001
Aug 03 10:01:23 archlinux booster: udev event {Action:add KObj:/devices/0020:1022:0001.0002/hidraw/hidraw1 Env:map[ACTION:add DEVNAME:hidraw1 DEVPATH:/devices/0020:1022:00>
Aug 03 10:01:23 archlinux booster: udev event {Action:add KObj:/devices/0020:1022:0001.0002 Env:map[ACTION:add DEVPATH:/devices/0020:1022:0001.0002 HID_ID:0020:00001022:00>
Aug 03 10:01:23 archlinux booster: udev event {Action:add KObj:/module/nvme Env:map[ACTION:add DEVPATH:/module/nvme SEQNUM:1485 SUBSYSTEM:module]}
Aug 03 10:01:23 archlinux booster: udev event {Action:add KObj:/bus/pci/drivers/nvme Env:map[ACTION:add DEVPATH:/bus/pci/drivers/nvme SEQNUM:1484 SUBSYSTEM:drivers]}
Aug 03 10:01:23 archlinux booster: udev event {Action:bind KObj:/devices/pci0000:00/0000:00:02.4/0000:02:00.0 Env:map[ACTION:bind DEVPATH:/devices/pci0000:00/0000:00:02.4/>
Aug 03 10:01:23 archlinux booster: udev event {Action:add KObj:/devices/pci0000:00/0000:00:02.4/0000:02:00.0/nvme/nvme0 Env:map[ACTION:add DEVNAME:nvme0 DEVPATH:/devices/p>
Aug 03 10:01:23 archlinux booster: udev event {Action:remove KObj:/devices/platform/AMDI0010:03/i2c-0/i2c-ELAN074E:00/0018:04F3:31B5.0001/input/input5/event4 Env:map[ACTIO>
Aug 03 10:01:23 archlinux booster: udev event {Action:remove KObj:/devices/platform/AMDI0010:03/i2c-0/i2c-ELAN074E:00/0018:04F3:31B5.0001/input/input3 Env:map[ACTION:remov>
Aug 03 10:01:23 archlinux booster: udev event {Action:add KObj:/module/nvme_core Env:map[ACTION:add DEVPATH:/module/nvme_core SEQNUM:1479 SUBSYSTEM:module]}
Aug 03 10:01:23 archlinux booster: udev event {Action:add KObj:/class/nvme-generic Env:map[ACTION:add DEVPATH:/class/nvme-generic SEQNUM:1478 SUBSYSTEM:class]}
Aug 03 10:01:23 archlinux booster: udev event {Action:add KObj:/class/nvme-subsystem Env:map[ACTION:add DEVPATH:/class/nvme-subsystem SEQNUM:1477 SUBSYSTEM:class]}
Aug 03 10:01:23 archlinux booster: udev event {Action:add KObj:/class/nvme Env:map[ACTION:add DEVPATH:/class/nvme SEQNUM:1476 SUBSYSTEM:class]}
Aug 03 10:01:23 archlinux booster: udev event {Action:add KObj:/devices/virtual/workqueue/nvme-delete-wq Env:map[ACTION:add DEVPATH:/devices/virtual/workqueue/nvme-delete->
Aug 03 10:01:23 archlinux booster: udev event {Action:add KObj:/devices/virtual/workqueue/nvme-reset-wq Env:map[ACTION:add DEVPATH:/devices/virtual/workqueue/nvme-reset-wq>
Aug 03 10:01:23 archlinux booster: udev event {Action:add KObj:/devices/virtual/workqueue/nvme-wq Env:map[ACTION:add DEVPATH:/devices/virtual/workqueue/nvme-wq SEQNUM:1473>
Aug 03 10:01:23 archlinux booster: udev event {Action:add KObj:/module/aesni_intel Env:map[ACTION:add DEVPATH:/module/aesni_intel SEQNUM:1472 SUBSYSTEM:module]}
Aug 03 10:01:23 archlinux booster: udev event {Action:add KObj:/module/polyval_clmulni Env:map[ACTION:add DEVPATH:/module/polyval_clmulni SEQNUM:1471 SUBSYSTEM:module]}
Aug 03 10:01:23 archlinux booster: udev event {Action:add KObj:/module/xhci_pci Env:map[ACTION:add DEVPATH:/module/xhci_pci SEQNUM:1470 SUBSYSTEM:module]}
Aug 03 10:01:23 archlinux booster: udev event {Action:add KObj:/bus/pci/drivers/xhci_hcd Env:map[ACTION:add DEVPATH:/bus/pci/drivers/xhci_hcd SEQNUM:1469 SUBSYSTEM:drivers>
Aug 03 10:01:23 archlinux booster: udev event {Action:bind KObj:/devices/pci0000:00/0000:00:08.1/0000:03:00.4 Env:map[ACTION:bind DEVPATH:/devices/pci0000:00/0000:00:08.1/>
Aug 03 10:01:23 archlinux booster: udev event {Action:bind KObj:/devices/pci0000:00/0000:00:08.1/0000:03:00.4/usb4 Env:map[ACTION:bind BUSNUM:004 DEVNAME:bus/usb/004/001 D>
Aug 03 10:01:23 archlinux booster: udev event {Action:bind KObj:/devices/pci0000:00/0000:00:08.1/0000:03:00.4/usb4/4-0:1.0 Env:map[ACTION:bind DEVPATH:/devices/pci0000:00/>
Aug 03 10:01:23 archlinux booster: udev event {Action:add KObj:/devices/pci0000:00/0000:00:08.1/0000:03:00.4/usb4/4-0:1.0 Env:map[ACTION:add DEVPATH:/devices/pci0000:00/00>
Aug 03 10:01:23 archlinux booster: udev event {Action:add KObj:/devices/pci0000:00/0000:00:08.1/0000:03:00.4/usb4 Env:map[ACTION:add BUSNUM:004 DEVNAME:bus/usb/004/001 DEV>
Aug 03 10:01:23 archlinux kernel:  nvme0n1: p1 p2
Aug 03 10:01:23 archlinux kernel: nvme nvme0: Ignoring bogus Namespace Identifiers
Aug 03 10:01:23 archlinux booster: udev event {Action:bind KObj:/devices/pci0000:00/0000:00:08.1/0000:03:00.4/usb3 Env:map[ACTION:bind BUSNUM:003 DEVNAME:bus/usb/003/001 D>
Aug 03 10:01:23 archlinux booster: udev event {Action:bind KObj:/devices/pci0000:00/0000:00:08.1/0000:03:00.4/usb3/3-0:1.0 Env:map[ACTION:bind DEVPATH:/devices/pci0000:00/>
Aug 03 10:01:23 archlinux kernel: nvme nvme0: 16/0/0 default/read/poll queues
Aug 03 10:01:23 archlinux booster: udev event {Action:add KObj:/devices/pci0000:00/0000:00:08.1/0000:03:00.4/usb3/3-0:1.0 Env:map[ACTION:add DEVPATH:/devices/pci0000:00/00>
Aug 03 10:01:23 archlinux booster: udev event {Action:add KObj:/devices/pci0000:00/0000:00:08.1/0000:03:00.4/usb3 Env:map[ACTION:add BUSNUM:003 DEVNAME:bus/usb/003/001 DEV>
Aug 03 10:01:23 archlinux booster: udev event {Action:add KObj:/devices/pci0000:00/0000:00:08.1/0000:03:00.4/wakeup/wakeup46 Env:map[ACTION:add DEVPATH:/devices/pci0000:00>
Aug 03 10:01:23 archlinux booster: udev event {Action:bind KObj:/devices/pci0000:00/0000:00:08.1/0000:03:00.3 Env:map[ACTION:bind DEVPATH:/devices/pci0000:00/0000:00:08.1/>
Aug 03 10:01:23 archlinux booster: udev event {Action:bind KObj:/devices/pci0000:00/0000:00:08.1/0000:03:00.3/usb2 Env:map[ACTION:bind BUSNUM:002 DEVNAME:bus/usb/002/001 D>
Aug 03 10:01:23 archlinux booster: no match found for alias usb:v1D6Bp0003d0601dc09dsc00dp03ic09isc00ip00in00
Aug 03 10:01:23 archlinux booster: udev event {Action:bind KObj:/devices/pci0000:00/0000:00:08.1/0000:03:00.3/usb2/2-0:1.0 Env:map[ACTION:bind DEVPATH:/devices/pci0000:00/>
Aug 03 10:01:23 archlinux booster: udev event {Action:add KObj:/devices/pci0000:00/0000:00:08.1/0000:03:00.3/usb2/2-0:1.0 Env:map[ACTION:add DEVPATH:/devices/pci0000:00/00>
Aug 03 10:01:23 archlinux booster: udev event {Action:add KObj:/devices/pci0000:00/0000:00:08.1/0000:03:00.3/usb2 Env:map[ACTION:add BUSNUM:002 DEVNAME:bus/usb/002/001 DEV>
Aug 03 10:01:23 archlinux booster: udev event {Action:bind KObj:/devices/pci0000:00/0000:00:08.1/0000:03:00.3/usb1 Env:map[ACTION:bind BUSNUM:001 DEVNAME:bus/usb/001/001 D>
Aug 03 10:01:23 archlinux booster: no match found for alias usb:v1D6Bp0002d0601dc09dsc00dp01ic09isc00ip00in00
Aug 03 10:01:23 archlinux booster: udev event {Action:bind KObj:/devices/pci0000:00/0000:00:08.1/0000:03:00.3/usb1/1-0:1.0 Env:map[ACTION:bind DEVPATH:/devices/pci0000:00/>
Aug 03 10:01:23 archlinux booster: udev event {Action:add KObj:/devices/pci0000:00/0000:00:08.1/0000:03:00.3/usb1/1-0:1.0 Env:map[ACTION:add DEVPATH:/devices/pci0000:00/00>
Aug 03 10:01:23 archlinux booster: udev event {Action:add KObj:/devices/pci0000:00/0000:00:08.1/0000:03:00.3/usb1 Env:map[ACTION:add BUSNUM:001 DEVNAME:bus/usb/001/001 DEV>
Aug 03 10:01:23 archlinux booster: udev event {Action:add KObj:/devices/pci0000:00/0000:00:08.1/0000:03:00.3/wakeup/wakeup45 Env:map[ACTION:add DEVPATH:/devices/pci0000:00>
Aug 03 10:01:23 archlinux booster: udev event {Action:add KObj:/module/ghash_clmulni_intel Env:map[ACTION:add DEVPATH:/module/ghash_clmulni_intel SEQNUM:1448 SUBSYSTEM:mod>
Aug 03 10:01:23 archlinux kernel: hid-multitouch 0018:04F3:31B5.0001: input,hidraw0: I2C HID v1.00 Mouse [ELAN074E:00 04F3:31B5] on i2c-ELAN074E:00
Aug 03 10:01:23 archlinux kernel: input: ELAN074E:00 04F3:31B5 Touchpad as /devices/platform/AMDI0010:03/i2c-0/i2c-ELAN074E:00/0018:04F3:31B5.0001/input/input8
Aug 03 10:01:23 archlinux kernel: input: ELAN074E:00 04F3:31B5 Mouse as /devices/platform/AMDI0010:03/i2c-0/i2c-ELAN074E:00/0018:04F3:31B5.0001/input/input6
Aug 03 10:01:23 archlinux kernel: pcie_mp2_amd: probe of 0000:03:00.7 failed with error -95
Aug 03 10:01:23 archlinux kernel: pcie_mp2_amd 0000:03:00.7: Failed to discover, sensors not enabled is 1
Aug 03 10:01:23 archlinux booster: udev event {Action:add KObj:/module/crypto_simd Env:map[ACTION:add DEVPATH:/module/crypto_simd SEQNUM:1447 SUBSYSTEM:module]}
Aug 03 10:01:23 archlinux booster: loading module vivaldi_fmap
Aug 03 10:01:23 archlinux booster: loading module libps2
Aug 03 10:01:23 archlinux booster: loading module serio_raw
Aug 03 10:01:23 archlinux booster: modalias serio:ty06pr00id00ex00 matched module atkbd
Aug 03 10:01:23 archlinux booster: udev event {Action:add KObj:/module/polyval_generic Env:map[ACTION:add DEVPATH:/module/polyval_generic SEQNUM:1446 SUBSYSTEM:module]}
Aug 03 10:01:23 archlinux kernel: hid-generic 0020:1022:0001.0004: hidraw3: SENSOR HUB HID v0.00 Device [hid-amdsfh 1022:0001] on pcie_mp2_amd
Aug 03 10:01:23 archlinux booster: udev event {Action:add KObj:/module/i8042 Env:map[ACTION:add DEVPATH:/module/i8042 SEQNUM:1445 SUBSYSTEM:module]}
Aug 03 10:01:23 archlinux booster: modalias serio:ty06pr00id00ex00 matched module serio_raw
Aug 03 10:01:23 archlinux booster: udev event {Action:bind KObj:/devices/platform/i8042 Env:map[ACTION:bind DEVPATH:/devices/platform/i8042 DRIVER:i8042 MODALIAS:platform:>
Aug 03 10:01:23 archlinux kernel: hid-generic 0020:1022:0001.0003: hidraw2: SENSOR HUB HID v0.00 Device [hid-amdsfh 1022:0001] on pcie_mp2_amd
Aug 03 10:01:23 archlinux booster: udev event {Action:add KObj:/devices/platform/i8042/serio0 Env:map[ACTION:add DEVPATH:/devices/platform/i8042/serio0 MODALIAS:serio:ty06>
Aug 03 10:01:23 archlinux kernel: hid-generic 0020:1022:0001.0002: hidraw1: SENSOR HUB HID v0.00 Device [hid-amdsfh 1022:0001] on pcie_mp2_amd
Aug 03 10:01:23 archlinux booster: udev event {Action:add KObj:/devices/platform/i8042/serio0/wakeup/wakeup43 Env:map[ACTION:add DEVPATH:/devices/platform/i8042/serio0/wak>
Aug 03 10:01:23 archlinux booster: udev event {Action:add KObj:/module/nvme_common Env:map[ACTION:add DEVPATH:/module/nvme_common SEQNUM:1441 SUBSYSTEM:module]}
Aug 03 10:01:23 archlinux booster: no match found for alias platform:i8042
Aug 03 10:01:23 archlinux booster: udev event {Action:add KObj:/devices/platform/i8042 Env:map[ACTION:add DEVPATH:/devices/platform/i8042 MODALIAS:platform:i8042 SEQNUM:14>
Aug 03 10:01:23 archlinux kernel: nvme nvme0: pci function 0000:02:00.0
Aug 03 10:01:23 archlinux kernel: nvme 0000:02:00.0: platform quirk: setting simple suspend
Aug 03 10:01:23 archlinux booster: udev event {Action:add KObj:/bus/platform/drivers/i8042 Env:map[ACTION:add DEVPATH:/bus/platform/drivers/i8042 SEQNUM:1439 SUBSYSTEM:dri>
Aug 03 10:01:23 archlinux booster: udev event {Action:add KObj:/bus/pnp/drivers/i8042 aux Env:map[ACTION:add DEVPATH:/bus/pnp/drivers/i8042 aux SEQNUM:1438 SUBSYSTEM:drive>
Aug 03 10:01:23 archlinux booster: udev event {Action:add KObj:/bus/pnp/drivers/i8042 kbd Env:map[ACTION:add DEVPATH:/bus/pnp/drivers/i8042 kbd SEQNUM:1437 SUBSYSTEM:drive>
Aug 03 10:01:23 archlinux booster: udev event {Action:bind KObj:/devices/pnp0/00:02 Env:map[ACTION:bind DEVPATH:/devices/pnp0/00:02 DRIVER:i8042 kbd SEQNUM:1436 SUBSYSTEM:>
Aug 03 10:01:23 archlinux booster: loading module nvme
Aug 03 10:01:23 archlinux booster: udev event {Action:remove KObj:/devices/platform/AMDI0010:03/i2c-0/i2c-ELAN074E:00/0018:04F3:31B5.0001/input/input3/event3 Env:map[ACTIO>
Aug 03 10:01:23 archlinux booster: udev event {Action:add KObj:/bus/hid/drivers/hid-multitouch Env:map[ACTION:add DEVPATH:/bus/hid/drivers/hid-multitouch SEQNUM:1434 SUBSY>
Aug 03 10:01:23 archlinux kernel: AES CTR mode by8 optimization enabled
Aug 03 10:01:23 archlinux kernel: AVX2 version of gcm_enc/dec engaged.
Aug 03 10:01:23 archlinux booster: udev event {Action:add KObj:/module/xhci_pci_renesas Env:map[ACTION:add DEVPATH:/module/xhci_pci_renesas SEQNUM:1433 SUBSYSTEM:module]}
Aug 03 10:01:23 archlinux booster: udev event {Action:add KObj:/module/cryptd Env:map[ACTION:add DEVPATH:/module/cryptd SEQNUM:1432 SUBSYSTEM:module]}
Aug 03 10:01:23 archlinux kernel: hub 4-0:1.0: 2 ports detected
Aug 03 10:01:23 archlinux kernel: hub 4-0:1.0: USB hub found
Aug 03 10:01:23 archlinux kernel: usb usb4: SerialNumber: 0000:03:00.4
Aug 03 10:01:23 archlinux kernel: usb usb4: Manufacturer: Linux 6.1.39-3-lts xhci-hcd
Aug 03 10:01:23 archlinux kernel: usb usb4: Product: xHCI Host Controller
Aug 03 10:01:23 archlinux kernel: usb usb4: New USB device strings: Mfr=3, Product=2, SerialNumber=1
Aug 03 10:01:23 archlinux kernel: usb usb4: New USB device found, idVendor=1d6b, idProduct=0003, bcdDevice= 6.01
Aug 03 10:01:23 archlinux kernel: usb usb4: We don't know the algorithms for LPM for this host, disabling LPM.
Aug 03 10:01:23 archlinux kernel: hub 3-0:1.0: 4 ports detected
Aug 03 10:01:23 archlinux kernel: hub 3-0:1.0: USB hub found
Aug 03 10:01:23 archlinux kernel: usb usb3: SerialNumber: 0000:03:00.4
Aug 03 10:01:23 archlinux kernel: usb usb3: Manufacturer: Linux 6.1.39-3-lts xhci-hcd
Aug 03 10:01:23 archlinux kernel: usb usb3: Product: xHCI Host Controller
Aug 03 10:01:23 archlinux kernel: usb usb3: New USB device strings: Mfr=3, Product=2, SerialNumber=1
Aug 03 10:01:23 archlinux kernel: usb usb3: New USB device found, idVendor=1d6b, idProduct=0002, bcdDevice= 6.01
Aug 03 10:01:23 archlinux kernel: xhci_hcd 0000:03:00.4: Host supports USB 3.1 Enhanced SuperSpeed
Aug 03 10:01:23 archlinux kernel: xhci_hcd 0000:03:00.4: new USB bus registered, assigned bus number 4
Aug 03 10:01:23 archlinux kernel: xhci_hcd 0000:03:00.4: xHCI Host Controller
Aug 03 10:01:23 archlinux kernel: xhci_hcd 0000:03:00.4: hcc params 0x0268ffe5 hci version 0x110 quirks 0x0000020000000410
Aug 03 10:01:23 archlinux kernel: xhci_hcd 0000:03:00.4: new USB bus registered, assigned bus number 3
Aug 03 10:01:23 archlinux kernel: xhci_hcd 0000:03:00.4: xHCI Host Controller
Aug 03 10:01:23 archlinux booster: udev event {Action:add KObj:/module/crc32c_intel Env:map[ACTION:add DEVPATH:/module/crc32c_intel SEQNUM:1431 SUBSYSTEM:module]}
Aug 03 10:01:23 archlinux kernel: hub 2-0:1.0: 2 ports detected
Aug 03 10:01:23 archlinux kernel: hub 2-0:1.0: USB hub found
Aug 03 10:01:23 archlinux kernel: usb usb2: SerialNumber: 0000:03:00.3
Aug 03 10:01:23 archlinux kernel: usb usb2: Manufacturer: Linux 6.1.39-3-lts xhci-hcd
Aug 03 10:01:23 archlinux kernel: usb usb2: Product: xHCI Host Controller
Aug 03 10:01:23 archlinux kernel: usb usb2: New USB device strings: Mfr=3, Product=2, SerialNumber=1
Aug 03 10:01:23 archlinux kernel: usb usb2: New USB device found, idVendor=1d6b, idProduct=0003, bcdDevice= 6.01
Aug 03 10:01:23 archlinux kernel: usb usb2: We don't know the algorithms for LPM for this host, disabling LPM.
Aug 03 10:01:23 archlinux kernel: hub 1-0:1.0: 4 ports detected
Aug 03 10:01:23 archlinux kernel: hub 1-0:1.0: USB hub found
Aug 03 10:01:23 archlinux booster: udev event {Action:add KObj:/module/crc32_pclmul Env:map[ACTION:add DEVPATH:/module/crc32_pclmul SEQNUM:1430 SUBSYSTEM:module]}
Aug 03 10:01:23 archlinux kernel: usb usb1: SerialNumber: 0000:03:00.3
Aug 03 10:01:23 archlinux kernel: usb usb1: Manufacturer: Linux 6.1.39-3-lts xhci-hcd
Aug 03 10:01:23 archlinux kernel: usb usb1: Product: xHCI Host Controller
Aug 03 10:01:23 archlinux kernel: usb usb1: New USB device strings: Mfr=3, Product=2, SerialNumber=1
Aug 03 10:01:23 archlinux kernel: usb usb1: New USB device found, idVendor=1d6b, idProduct=0002, bcdDevice= 6.01
Aug 03 10:01:23 archlinux kernel: xhci_hcd 0000:03:00.3: Host supports USB 3.1 Enhanced SuperSpeed
Aug 03 10:01:23 archlinux kernel: xhci_hcd 0000:03:00.3: new USB bus registered, assigned bus number 2
Aug 03 10:01:23 archlinux kernel: xhci_hcd 0000:03:00.3: xHCI Host Controller
Aug 03 10:01:23 archlinux kernel: xhci_hcd 0000:03:00.3: hcc params 0x0268ffe5 hci version 0x110 quirks 0x0000020000000410
Aug 03 10:01:23 archlinux kernel: xhci_hcd 0000:03:00.3: new USB bus registered, assigned bus number 1
Aug 03 10:01:23 archlinux kernel: xhci_hcd 0000:03:00.3: xHCI Host Controller
Aug 03 10:01:23 archlinux booster: loading module aesni_intel
Aug 03 10:01:23 archlinux booster: loading module polyval_clmulni
Aug 03 10:01:23 archlinux booster: loading module nvme_core
Aug 03 10:01:23 archlinux booster: udev event {Action:add KObj:/module/crct10dif_pclmul Env:map[ACTION:add DEVPATH:/module/crct10dif_pclmul SEQNUM:1429 SUBSYSTEM:module]}
Aug 03 10:01:23 archlinux kernel: serio: i8042 KBD port at 0x60,0x64 irq 1
Aug 03 10:01:23 archlinux booster: udev event {Action:add KObj:/module/gf128mul Env:map[ACTION:add DEVPATH:/module/gf128mul SEQNUM:1428 SUBSYSTEM:module]}
Aug 03 10:01:23 archlinux booster: udev event {Action:add KObj:/module/sha512_ssse3 Env:map[ACTION:add DEVPATH:/module/sha512_ssse3 SEQNUM:1427 SUBSYSTEM:module]}
Aug 03 10:01:23 archlinux kernel: i8042: PNP: PS/2 appears to have AUX port disabled, if this is incorrect please boot with i8042.nopnp
Aug 03 10:01:23 archlinux kernel: i8042: PNP: PS/2 Controller [PNP0303:KBD0] at 0x60,0x64 irq 1
Aug 03 10:01:23 archlinux kernel: pcie_mp2_amd 0000:03:00.7: enabling device (0000 -> 0002)
Aug 03 10:01:23 archlinux booster: loading module xhci_pci
Aug 03 10:01:23 archlinux booster: udev event {Action:add KObj:/module/serio Env:map[ACTION:add DEVPATH:/module/serio SEQNUM:1426 SUBSYSTEM:module]}
Aug 03 10:01:23 archlinux booster: loading module crypto_simd
Aug 03 10:01:23 archlinux booster: loading module ghash_clmulni_intel
Aug 03 10:01:23 archlinux kernel: cryptd: max_cpu_qlen set to 1000
Aug 03 10:01:23 archlinux booster: modalias pci:v00001002d00001638sv0000103Csd00008919bc03sc00i00 matched module amdgpu
Aug 03 10:01:23 archlinux booster: loading module polyval_generic
Aug 03 10:01:23 archlinux booster: loading module i8042
Aug 03 10:01:23 archlinux booster: udev event {Action:add KObj:/bus/serio Env:map[ACTION:add DEVPATH:/bus/serio SEQNUM:1425 SUBSYSTEM:bus]}
Aug 03 10:01:23 archlinux booster: loading module hid_multitouch
Aug 03 10:01:23 archlinux booster: loading module amd_sfh
Aug 03 10:01:23 archlinux booster: loading module xhci_pci_renesas
Aug 03 10:01:23 archlinux booster: loading module nvme_common
Aug 03 10:01:23 archlinux booster: modalias pci:v00001C5Cd0000174Asv00001C5Csd0000174Abc01sc08i02 matched module nvme
Aug 03 10:01:23 archlinux booster: loading module crc32_pclmul
Aug 03 10:01:23 archlinux booster: loading module cryptd
Aug 03 10:01:23 archlinux booster: loading module crc32c_intel
Aug 03 10:01:23 archlinux booster: loading module crct10dif_pclmul
Aug 03 10:01:23 archlinux booster: loading module gf128mul
Aug 03 10:01:23 archlinux booster: loading module sha512_ssse3
Aug 03 10:01:23 archlinux booster: modalias cpu:type:x86,ven0002fam0019mod0050:feature:,0000,0001,0002,0003,0004,0005,0006,0007,0008,0009,000B,000C,000D,000E,000F,0010,001>
Aug 03 10:01:23 archlinux booster: modalias cpu:type:x86,ven0002fam0019mod0050:feature:,0000,0001,0002,0003,0004,0005,0006,0007,0008,0009,000B,000C,000D,000E,000F,0010,001>
Aug 03 10:01:23 archlinux booster: modalias cpu:type:x86,ven0002fam0019mod0050:feature:,0000,0001,0002,0003,0004,0005,0006,0007,0008,0009,000B,000C,000D,000E,000F,0010,001>
Aug 03 10:01:23 archlinux booster: modalias cpu:type:x86,ven0002fam0019mod0050:feature:,0000,0001,0002,0003,0004,0005,0006,0007,0008,0009,000B,000C,000D,000E,000F,0010,001>
Aug 03 10:01:23 archlinux booster: modalias cpu:type:x86,ven0002fam0019mod0050:feature:,0000,0001,0002,0003,0004,0005,0006,0007,0008,0009,000B,000C,000D,000E,000F,0010,001>
Aug 03 10:01:23 archlinux booster: modalias cpu:type:x86,ven0002fam0019mod0050:feature:,0000,0001,0002,0003,0004,0005,0006,0007,0008,0009,000B,000C,000D,000E,000F,0010,001>
Aug 03 10:01:23 archlinux booster: modalias cpu:type:x86,ven0002fam0019mod0050:feature:,0000,0001,0002,0003,0004,0005,0006,0007,0008,0009,000B,000C,000D,000E,000F,0010,001>
Aug 03 10:01:23 archlinux booster: modalias cpu:type:x86,ven0002fam0019mod0050:feature:,0000,0001,0002,0003,0004,0005,0006,0007,0008,0009,000B,000C,000D,000E,000F,0010,001>
Aug 03 10:01:23 archlinux booster: loading module serio
</pre>
</details>

booster.yaml
```
modules_force_load: amdgpu,usbhid,hid_sensor_hub
extra_files: fido2-assert
universal: true
```
<details>
<summary>Universal</summary>
<pre>
Aug 03 09:44:08 archlinux booster: HID hidraw1 supports FIDO, trying it to recover the password
Aug 03 09:44:08 archlinux booster: HID hidraw0 does not support FIDO
Aug 03 09:44:08 archlinux booster: loading module asn1_encoder
Aug 03 09:44:08 archlinux booster: loading module dm_mod
Aug 03 09:44:08 archlinux booster: loading module cbc
Aug 03 09:44:08 archlinux booster: loading module tee
Aug 03 09:44:08 archlinux booster: a mapping for LUKS device /dev/nvme0n1p2 has been found
Aug 03 09:44:08 archlinux booster: blkinfo for /dev/nvme0n1p1: type=fat UUID=dce4f928 LABEL=NO NAME
Aug 03 09:44:08 archlinux booster: blkinfo for /dev/nvme0n1p2: type=luks UUID=5e975a1e-da7a-424a-91c9-e58d441f9677 LABEL=
Aug 03 09:44:08 archlinux booster: found a new device /dev/nvme0n1p1
Aug 03 09:44:08 archlinux booster: found a new device /dev/nvme0n1p2
Aug 03 09:44:08 archlinux booster: blkinfo for /dev/nvme0n1: type=gpt UUID=5d41ee56-196a-4696-aefc-9663bca535e5 LABEL=
Aug 03 09:44:08 archlinux booster: loading module polyval_clmulni
Aug 03 09:44:08 archlinux booster: loading module atkbd
Aug 03 09:44:08 archlinux booster: udev event {Action:add KObj:/devices/pci0000:00/0000:00:02.4/0000:02:00.0/nvme/nvme0/ng0n1 Env:map[ACTION:add DEVNAME:ng0n>
Aug 03 09:44:08 archlinux booster: udev event {Action:add KObj:/devices/pci0000:00/0000:00:02.4/0000:02:00.0/nvme/nvme0/nvme0n1/nvme0n1p2 Env:map[ACTION:add >
Aug 03 09:44:08 archlinux booster: found a new device /dev/nvme0n1
Aug 03 09:44:08 archlinux booster: udev event {Action:add KObj:/devices/pci0000:00/0000:00:02.4/0000:02:00.0/nvme/nvme0/nvme0n1/nvme0n1p1 Env:map[ACTION:add >
Aug 03 09:44:08 archlinux booster: udev event {Action:add KObj:/devices/pci0000:00/0000:00:02.4/0000:02:00.0/nvme/nvme0/nvme0n1 Env:map[ACTION:add DEVNAME:nv>
Aug 03 09:44:08 archlinux kernel:  nvme0n1: p1 p2
Aug 03 09:44:08 archlinux booster: udev event {Action:add KObj:/devices/virtual/bdi/259:0 Env:map[ACTION:add DEVPATH:/devices/virtual/bdi/259:0 SEQNUM:1580 S>
Aug 03 09:44:08 archlinux kernel: nvme nvme0: Ignoring bogus Namespace Identifiers
Aug 03 09:44:08 archlinux booster: udev event {Action:change KObj:/devices/pci0000:00/0000:00:02.4/0000:02:00.0/nvme/nvme0 Env:map[ACTION:change DEVNAME:nvme>
Aug 03 09:44:08 archlinux kernel: nvme nvme0: 16/0/0 default/read/poll queues
Aug 03 09:44:08 archlinux booster: udev event {Action:add KObj:/devices/pci0000:00/0000:00:02.4/0000:02:00.0/nvme/nvme0/hwmon3 Env:map[ACTION:add DEVPATH:/de>
Aug 03 09:44:08 archlinux booster: udev event {Action:add KObj:/module/hid_multitouch Env:map[ACTION:add DEVPATH:/module/hid_multitouch SEQNUM:1577 SUBSYSTEM>
Aug 03 09:44:08 archlinux booster: udev event {Action:bind KObj:/devices/platform/AMDI0010:03/i2c-0/i2c-ELAN074E:00/0018:04F3:31B5.0001 Env:map[ACTION:bind D>
Aug 03 09:44:08 archlinux booster: no match found for alias input:b0018v04F3p31B5e0100_e0,1,3,4,k110,145,148,14A,14D,14E,14F,ra0,1,2F,35,36,37,39,m5,lsfw
Aug 03 09:44:08 archlinux booster: udev event {Action:add KObj:/devices/platform/AMDI0010:03/i2c-0/i2c-ELAN074E:00/0018:04F3:31B5.0001/hidraw/hidraw0 Env:map>
Aug 03 09:44:08 archlinux booster: udev event {Action:add KObj:/devices/platform/AMDI0010:03/i2c-0/i2c-ELAN074E:00/0018:04F3:31B5.0001/input/input8/event4 En>
Aug 03 09:44:08 archlinux booster: udev event {Action:add KObj:/devices/platform/AMDI0010:03/i2c-0/i2c-ELAN074E:00/0018:04F3:31B5.0001/input/input8 Env:map[A>
Aug 03 09:44:08 archlinux kernel: hid-multitouch 0018:04F3:31B5.0001: input,hidraw0: I2C HID v1.00 Mouse [ELAN074E:00 04F3:31B5] on i2c-ELAN074E:00
Aug 03 09:44:08 archlinux booster: udev event {Action:add KObj:/devices/platform/AMDI0010:03/i2c-0/i2c-ELAN074E:00/0018:04F3:31B5.0001/input/input6/event3 En>
Aug 03 09:44:08 archlinux kernel: input: ELAN074E:00 04F3:31B5 Touchpad as /devices/platform/AMDI0010:03/i2c-0/i2c-ELAN074E:00/0018:04F3:31B5.0001/input/inpu>
Aug 03 09:44:08 archlinux booster: udev event {Action:add KObj:/devices/platform/AMDI0010:03/i2c-0/i2c-ELAN074E:00/0018:04F3:31B5.0001/input/input6 Env:map[A>
Aug 03 09:44:08 archlinux kernel: input: ELAN074E:00 04F3:31B5 Mouse as /devices/platform/AMDI0010:03/i2c-0/i2c-ELAN074E:00/0018:04F3:31B5.0001/input/input6
Aug 03 09:44:08 archlinux booster: udev event {Action:unbind KObj:/devices/platform/AMDI0010:03/i2c-0/i2c-ELAN074E:00/0018:04F3:31B5.0001 Env:map[ACTION:unbi>
Aug 03 09:44:08 archlinux booster: udev event {Action:remove KObj:/devices/platform/AMDI0010:03/i2c-0/i2c-ELAN074E:00/0018:04F3:31B5.0001/hidraw/hidraw0 Env:>
Aug 03 09:44:08 archlinux booster: udev event {Action:remove KObj:/devices/platform/AMDI0010:03/i2c-0/i2c-ELAN074E:00/0018:04F3:31B5.0001/input/input5 Env:ma>
Aug 03 09:44:08 archlinux kernel: usb 1-3: new high-speed USB device number 3 using xhci_hcd
Aug 03 09:44:08 archlinux booster: udev event {Action:add KObj:/devices/virtual/nvme-subsystem/nvme-subsys0 Env:map[ACTION:add DEVPATH:/devices/virtual/nvme->
Aug 03 09:44:08 archlinux kernel: amdgpu 0000:03:00.0: amdgpu: Will use PSP to load VCN firmware
Aug 03 09:44:08 archlinux kernel: [drm] Found VCN firmware Version ENC: 1.20 DEC: 5 VEP: 0 Revision: 3
Aug 03 09:44:08 archlinux booster: no match found for alias platform:efi_framebuffer
Aug 03 09:44:08 archlinux kernel: [drm] Loading DMUB firmware via PSP: version=0x01010027
Aug 03 09:44:08 archlinux booster: udev event {Action:remove KObj:/devices/platform/efi-framebuffer.0 Env:map[ACTION:remove DEVPATH:/devices/platform/efi-fra>
Aug 03 09:44:08 archlinux booster: udev event {Action:unbind KObj:/devices/platform/efi-framebuffer.0 Env:map[ACTION:unbind DEVPATH:/devices/platform/efi-fra>
Aug 03 09:44:08 archlinux booster: udev event {Action:remove KObj:/devices/platform/efi-framebuffer.0/graphics/fb0 Env:map[ACTION:remove DEVNAME:fb0 DEVPATH:>
Aug 03 09:44:08 archlinux booster: udev event {Action:add KObj:/devices/pci0000:00/0000:00:08.1/0000:03:00.0/ip_discovery/die/0/274 Env:map[ACTION:add DEVPAT>
Aug 03 09:44:08 archlinux booster: udev event {Action:add KObj:/devices/pci0000:00/0000:00:08.1/0000:03:00.0/ip_discovery/die/0/272 Env:map[ACTION:add DEVPAT>
Aug 03 09:44:08 archlinux booster: udev event {Action:add KObj:/devices/pci0000:00/0000:00:08.1/0000:03:00.0/ip_discovery/die/0/271 Env:map[ACTION:add DEVPAT>
Aug 03 09:44:08 archlinux booster: udev event {Action:add KObj:/devices/pci0000:00/0000:00:08.1/0000:03:00.0/ip_discovery/die/0/255 Env:map[ACTION:add DEVPAT>
Aug 03 09:44:08 archlinux booster: udev event {Action:add KObj:/devices/pci0000:00/0000:00:08.1/0000:03:00.0/ip_discovery/die/0/170 Env:map[ACTION:add DEVPAT>
Aug 03 09:44:08 archlinux booster: udev event {Action:add KObj:/devices/pci0000:00/0000:00:08.1/0000:03:00.0/ip_discovery/die/0/150 Env:map[ACTION:add DEVPAT>
Aug 03 09:44:08 archlinux booster: udev event {Action:add KObj:/devices/pci0000:00/0000:00:08.1/0000:03:00.0/ip_discovery/die/0/128 Env:map[ACTION:add DEVPAT>
Aug 03 09:44:08 archlinux booster: udev event {Action:add KObj:/devices/pci0000:00/0000:00:08.1/0000:03:00.0/ip_discovery/die/0/124 Env:map[ACTION:add DEVPAT>
Aug 03 09:44:08 archlinux kernel: amdgpu 0000:03:00.0: amdgpu: PSP runtime database doesn't exist
Aug 03 09:44:08 archlinux booster: udev event {Action:add KObj:/devices/pci0000:00/0000:00:08.1/0000:03:00.0/ip_discovery/die/0/108 Env:map[ACTION:add DEVPAT>
Aug 03 09:44:08 archlinux kernel: amdgpu 0000:03:00.0: amdgpu: PSP runtime database doesn't exist
Aug 03 09:44:08 archlinux booster: udev event {Action:add KObj:/devices/pci0000:00/0000:00:08.1/0000:03:00.0/ip_discovery/die/0/100 Env:map[ACTION:add DEVPAT>
Aug 03 09:44:08 archlinux booster: no match found for alias hid:b0003g0001v00001050p00000402
Aug 03 09:44:08 archlinux booster: udev event {Action:add KObj:/devices/pci0000:00/0000:00:08.1/0000:03:00.0/ip_discovery/die/0/90 Env:map[ACTION:add DEVPATH>
Aug 03 09:44:08 archlinux booster: udev event {Action:add KObj:/devices/pci0000:00/0000:00:08.1/0000:03:00.0/ip_discovery/die/0/80 Env:map[ACTION:add DEVPATH>
Aug 03 09:44:08 archlinux booster: udev event {Action:add KObj:/devices/pci0000:00/0000:00:08.1/0000:03:00.0/ip_discovery/die/0/70 Env:map[ACTION:add DEVPATH>
Aug 03 09:44:08 archlinux booster: udev event {Action:add KObj:/devices/pci0000:00/0000:00:08.1/0000:03:00.0/ip_discovery/die/0/48 Env:map[ACTION:add DEVPATH>
Aug 03 09:44:08 archlinux booster: udev event {Action:add KObj:/devices/pci0000:00/0000:00:08.1/0000:03:00.0/ip_discovery/die/0/47 Env:map[ACTION:add DEVPATH>
Aug 03 09:44:08 archlinux booster: udev event {Action:add KObj:/devices/pci0000:00/0000:00:08.1/0000:03:00.0/ip_discovery/die/0/46 Env:map[ACTION:add DEVPATH>
Aug 03 09:44:08 archlinux booster: udev event {Action:add KObj:/devices/pci0000:00/0000:00:08.1/0000:03:00.0/ip_discovery/die/0/45 Env:map[ACTION:add DEVPATH>
Aug 03 09:44:08 archlinux booster: udev event {Action:add KObj:/devices/pci0000:00/0000:00:08.1/0000:03:00.0/ip_discovery/die/0/44 Env:map[ACTION:add DEVPATH>
Aug 03 09:44:08 archlinux booster: udev event {Action:add KObj:/devices/pci0000:00/0000:00:08.1/0000:03:00.0/ip_discovery/die/0/42 Env:map[ACTION:add DEVPATH>
Aug 03 09:44:08 archlinux booster: udev event {Action:add KObj:/devices/pci0000:00/0000:00:08.1/0000:03:00.0/ip_discovery/die/0/41 Env:map[ACTION:add DEVPATH>
Aug 03 09:44:08 archlinux booster: udev event {Action:add KObj:/devices/pci0000:00/0000:00:08.1/0000:03:00.0/ip_discovery/die/0/40 Env:map[ACTION:add DEVPATH>
Aug 03 09:44:08 archlinux booster: udev event {Action:add KObj:/devices/pci0000:00/0000:00:08.1/0000:03:00.0/ip_discovery/die/0/36 Env:map[ACTION:add DEVPATH>
Aug 03 09:44:08 archlinux booster: udev event {Action:add KObj:/devices/pci0000:00/0000:00:08.1/0000:03:00.0/ip_discovery/die/0/35 Env:map[ACTION:add DEVPATH>
Aug 03 09:44:08 archlinux booster: udev event {Action:add KObj:/devices/pci0000:00/0000:00:08.1/0000:03:00.0/ip_discovery/die/0/34 Env:map[ACTION:add DEVPATH>
Aug 03 09:44:08 archlinux booster: udev event {Action:add KObj:/devices/pci0000:00/0000:00:08.1/0000:03:00.0/ip_discovery/die/0/24 Env:map[ACTION:add DEVPATH>
Aug 03 09:44:08 archlinux booster: udev event {Action:add KObj:/devices/pci0000:00/0000:00:08.1/0000:03:00.0/ip_discovery/die/0/14 Env:map[ACTION:add DEVPATH>
Aug 03 09:44:08 archlinux booster: udev event {Action:add KObj:/devices/pci0000:00/0000:00:08.1/0000:03:00.0/ip_discovery/die/0/12 Env:map[ACTION:add DEVPATH>
Aug 03 09:44:08 archlinux booster: udev event {Action:add KObj:/devices/pci0000:00/0000:00:08.1/0000:03:00.0/ip_discovery/die/0/11 Env:map[ACTION:add DEVPATH>
Aug 03 09:44:08 archlinux booster: udev event {Action:add KObj:/devices/pci0000:00/0000:00:08.1/0000:03:00.0/ip_discovery/die/0/6 Env:map[ACTION:add DEVPATH:>
Aug 03 09:44:08 archlinux booster: udev event {Action:add KObj:/devices/pci0000:00/0000:00:08.1/0000:03:00.0/ip_discovery/die/0/5 Env:map[ACTION:add DEVPATH:>
Aug 03 09:44:08 archlinux booster: udev event {Action:add KObj:/devices/pci0000:00/0000:00:08.1/0000:03:00.0/ip_discovery/die/0/4 Env:map[ACTION:add DEVPATH:>
Aug 03 09:44:08 archlinux booster: udev event {Action:add KObj:/devices/pci0000:00/0000:00:08.1/0000:03:00.0/ip_discovery/die/0/3 Env:map[ACTION:add DEVPATH:>
Aug 03 09:44:08 archlinux booster: udev event {Action:add KObj:/devices/pci0000:00/0000:00:08.1/0000:03:00.0/ip_discovery/die/0/2 Env:map[ACTION:add DEVPATH:>
Aug 03 09:44:08 archlinux kernel: [drm] PTB located at 0x000000F41FC00000
Aug 03 09:44:08 archlinux kernel: [drm] PCIE GART of 1024M enabled.
Aug 03 09:44:08 archlinux booster: udev event {Action:add KObj:/devices/pci0000:00/0000:00:08.1/0000:03:00.0/ip_discovery/die/0/1 Env:map[ACTION:add DEVPATH:>
Aug 03 09:44:08 archlinux booster: udev event {Action:add KObj:/devices/pci0000:00/0000:00:08.1/0000:03:00.0/ip_discovery/die/0 Env:map[ACTION:add DEVPATH:/d>
Aug 03 09:44:08 archlinux booster: udev event {Action:remove KObj:/devices/platform/AMDI0010:03/i2c-0/i2c-ELAN074E:00/0018:04F3:31B5.0001/input/input5/event4>
Aug 03 09:44:08 archlinux booster: udev event {Action:remove KObj:/devices/platform/AMDI0010:03/i2c-0/i2c-ELAN074E:00/0018:04F3:31B5.0001/input/input3 Env:ma>
Aug 03 09:44:08 archlinux booster: udev event {Action:add KObj:/devices/virtual/kfd/kfd Env:map[ACTION:add DEVNAME:kfd DEVPATH:/devices/virtual/kfd/kfd MAJOR>
Aug 03 09:44:08 archlinux booster: udev event {Action:add KObj:/class/kfd Env:map[ACTION:add DEVPATH:/class/kfd SEQNUM:1525 SUBSYSTEM:class]}
Aug 03 09:44:08 archlinux kernel: [drm] GART: num cpu pages 262144, num gpu pages 262144
Aug 03 09:44:08 archlinux booster: udev event {Action:add KObj:/module/usbhid Env:map[ACTION:add DEVPATH:/module/usbhid SEQNUM:1524 SUBSYSTEM:module]}
Aug 03 09:44:08 archlinux kernel: [drm] amdgpu: 7668M of GTT memory ready.
Aug 03 09:44:08 archlinux kernel: [drm] amdgpu: 512M of VRAM memory ready
Aug 03 09:44:08 archlinux booster: udev event {Action:add KObj:/bus/usb/drivers/usbhid Env:map[ACTION:add DEVPATH:/bus/usb/drivers/usbhid SEQNUM:1523 SUBSYST>
Aug 03 09:44:08 archlinux kernel: [drm] RAM width 128bits DDR4
Aug 03 09:44:08 archlinux kernel: [drm] Detected VRAM RAM=512M, BAR=512M
Aug 03 09:44:08 archlinux booster: udev event {Action:bind KObj:/devices/pci0000:00/0000:00:08.1/0000:03:00.3/usb1/1-1/1-1:1.0 Env:map[ACTION:bind DEVPATH:/d>
Aug 03 09:44:08 archlinux kernel: amdgpu 0000:03:00.0: amdgpu: AGP: 267419648M 0x000000F800000000 - 0x0000FFFFFFFFFFFF
Aug 03 09:44:08 archlinux kernel: amdgpu 0000:03:00.0: amdgpu: GART: 1024M 0x0000000000000000 - 0x000000003FFFFFFF
Aug 03 09:44:08 archlinux kernel: amdgpu 0000:03:00.0: amdgpu: VRAM: 512M 0x000000F400000000 - 0x000000F41FFFFFFF (512M used)
Aug 03 09:44:08 archlinux kernel: [drm] vm size is 262144 GB, 4 levels, block size is 9-bit, fragment size is 9-bit
Aug 03 09:44:08 archlinux booster: udev event {Action:bind KObj:/devices/pci0000:00/0000:00:08.1/0000:03:00.3/usb1/1-1/1-1:1.0/0003:1050:0402.0005 Env:map[AC>
Aug 03 09:44:08 archlinux booster: udev event {Action:add KObj:/devices/pci0000:00/0000:00:08.1/0000:03:00.3/usb1/1-1/1-1:1.0/0003:1050:0402.0005/hidraw/hidr>
Aug 03 09:44:08 archlinux booster: udev event {Action:add KObj:/devices/pci0000:00/0000:00:08.1/0000:03:00.3/usb1/1-1/1-1:1.0/usbmisc/hiddev0 Env:map[ACTION:>
Aug 03 09:44:08 archlinux booster: udev event {Action:add KObj:/class/usbmisc Env:map[ACTION:add DEVPATH:/class/usbmisc SEQNUM:1518 SUBSYSTEM:class]}
Aug 03 09:44:08 archlinux booster: udev event {Action:add KObj:/devices/pci0000:00/0000:00:08.1/0000:03:00.3/usb1/1-1/1-1:1.0/0003:1050:0402.0005 Env:map[ACT>
Aug 03 09:44:08 archlinux booster: udev event {Action:add KObj:/module/aesni_intel Env:map[ACTION:add DEVPATH:/module/aesni_intel SEQNUM:1516 SUBSYSTEM:modul>
Aug 03 09:44:08 archlinux booster: udev event {Action:add KObj:/module/serio_raw Env:map[ACTION:add DEVPATH:/module/serio_raw SEQNUM:1515 SUBSYSTEM:module]}
Aug 03 09:44:08 archlinux booster: udev event {Action:add KObj:/module/libps2 Env:map[ACTION:add DEVPATH:/module/libps2 SEQNUM:1514 SUBSYSTEM:module]}
Aug 03 09:44:08 archlinux booster: udev event {Action:add KObj:/bus/serio/drivers/serio_raw Env:map[ACTION:add DEVPATH:/bus/serio/drivers/serio_raw SEQNUM:15>
Aug 03 09:44:08 archlinux booster: udev event {Action:add KObj:/module/vivaldi_fmap Env:map[ACTION:add DEVPATH:/module/vivaldi_fmap SEQNUM:1512 SUBSYSTEM:mod>
Aug 03 09:44:08 archlinux booster: udev event {Action:add KObj:/module/hid_sensor_hub Env:map[ACTION:add DEVPATH:/module/hid_sensor_hub SEQNUM:1511 SUBSYSTEM>
Aug 03 09:44:08 archlinux booster: udev event {Action:remove KObj:/devices/platform/AMDI0010:03/i2c-0/i2c-ELAN074E:00/0018:04F3:31B5.0001/input/input3/event3>
Aug 03 09:44:08 archlinux booster: udev event {Action:add KObj:/bus/hid/drivers/hid-multitouch Env:map[ACTION:add DEVPATH:/bus/hid/drivers/hid-multitouch SEQ>
Aug 03 09:44:08 archlinux booster: udev event {Action:add KObj:/module/nvme Env:map[ACTION:add DEVPATH:/module/nvme SEQNUM:1508 SUBSYSTEM:module]}
Aug 03 09:44:08 archlinux booster: udev event {Action:add KObj:/module/polyval_generic Env:map[ACTION:add DEVPATH:/module/polyval_generic SEQNUM:1507 SUBSYST>
Aug 03 09:44:08 archlinux kernel: amdgpu 0000:03:00.0: amdgpu: MODE2 reset
Aug 03 09:44:08 archlinux kernel: amdgpu 0000:03:00.0: amdgpu: Trusted Memory Zone (TMZ) feature enabled
Aug 03 09:44:08 archlinux kernel: amdgpu 0000:03:00.0: vgaarb: deactivate vga console
Aug 03 09:44:08 archlinux booster: udev event {Action:add KObj:/bus/pci/drivers/nvme Env:map[ACTION:add DEVPATH:/bus/pci/drivers/nvme SEQNUM:1506 SUBSYSTEM:d>
Aug 03 09:44:08 archlinux kernel: Console: switching to colour dummy device 80x25
Aug 03 09:44:08 archlinux kernel: [drm] JPEG decode is enabled in VM mode
Aug 03 09:44:08 archlinux kernel: [drm] VCN encode is enabled in VM mode
Aug 03 09:44:08 archlinux kernel: [drm] VCN decode is enabled in VM mode
Aug 03 09:44:08 archlinux kernel: amdgpu: ATOM BIOS: 113-CEZANNE-018
Aug 03 09:44:08 archlinux kernel: amdgpu 0000:03:00.0: amdgpu: Fetched VBIOS from VFCT
Aug 03 09:44:08 archlinux kernel: [drm] add ip block number 9 <jpeg_v2_0>
Aug 03 09:44:08 archlinux kernel: [drm] add ip block number 8 <vcn_v2_0>
Aug 03 09:44:08 archlinux kernel: [drm] add ip block number 7 <sdma_v4_0>
Aug 03 09:44:08 archlinux kernel: [drm] add ip block number 6 <gfx_v9_0>
Aug 03 09:44:08 archlinux kernel: [drm] add ip block number 5 <dm>
Aug 03 09:44:08 archlinux kernel: [drm] add ip block number 4 <smu>
Aug 03 09:44:08 archlinux kernel: [drm] add ip block number 3 <psp>
Aug 03 09:44:08 archlinux kernel: [drm] add ip block number 2 <vega10_ih>
Aug 03 09:44:08 archlinux kernel: [drm] add ip block number 1 <gmc_v9_0>
Aug 03 09:44:08 archlinux kernel: [drm] add ip block number 0 <soc15_common>
Aug 03 09:44:08 archlinux booster: udev event {Action:bind KObj:/devices/pci0000:00/0000:00:02.4/0000:02:00.0 Env:map[ACTION:bind DEVPATH:/devices/pci0000:00>
Aug 03 09:44:08 archlinux kernel: [drm] register mmio size: 524288
Aug 03 09:44:08 archlinux kernel: [drm] register mmio base: 0xFCD00000
Aug 03 09:44:08 archlinux booster: udev event {Action:add KObj:/devices/pci0000:00/0000:00:02.4/0000:02:00.0/nvme/nvme0 Env:map[ACTION:add DEVNAME:nvme0 DEVP>
Aug 03 09:44:08 archlinux kernel: [drm] initializing kernel modesetting (RENOIR 0x1002:0x1638 0x103C:0x8919 0xC2).
Aug 03 09:44:08 archlinux kernel: amdgpu 0000:03:00.0: enabling device (0006 -> 0007)
Aug 03 09:44:08 archlinux kernel: amdgpu: Topology: Add CPU node
Aug 03 09:44:08 archlinux kernel: amdgpu: Virtual CRAT table created for CPU
Aug 03 09:44:08 archlinux booster: udev event {Action:add KObj:/bus/hid/drivers/hid-sensor-hub Env:map[ACTION:add DEVPATH:/bus/hid/drivers/hid-sensor-hub SEQ>
Aug 03 09:44:08 archlinux kernel: usbhid: USB HID core driver
Aug 03 09:44:08 archlinux kernel: usbcore: registered new interface driver usbhid
Aug 03 09:44:08 archlinux kernel: hid-generic 0003:1050:0402.0005: hiddev96,hidraw1: USB HID v1.10 Device [Yubico YubiKey FIDO] on usb-0000:03:00.3-1/input0
Aug 03 09:44:08 archlinux kernel: [drm] amdgpu kernel modesetting enabled.
Aug 03 09:44:08 archlinux booster: no match found for alias usb:v0BDAp2852d0000dcE0dsc01dp01icE0isc01ip01in01
Aug 03 09:44:08 archlinux booster: udev event {Action:bind KObj:/devices/pci0000:00/0000:00:08.1/0000:03:00.4/usb3/3-3 Env:map[ACTION:bind BUSNUM:003 DEVNAME>
Aug 03 09:44:08 archlinux kernel: AES CTR mode by8 optimization enabled
Aug 03 09:44:08 archlinux kernel: AVX2 version of gcm_enc/dec engaged.
Aug 03 09:44:08 archlinux booster: loading module usbhid
Aug 03 09:44:08 archlinux kernel: nvme nvme0: pci function 0000:02:00.0
Aug 03 09:44:08 archlinux kernel: nvme 0000:02:00.0: platform quirk: setting simple suspend
Aug 03 09:44:08 archlinux kernel: clocksource: Switched to clocksource hpet
Aug 03 09:44:08 archlinux kernel: clocksource: Checking clocksource tsc synchronization from CPU 3 to CPUs 0,5,8,11.
Aug 03 09:44:08 archlinux booster: udev event {Action:add KObj:/devices/pci0000:00/0000:00:08.1/0000:03:00.4/usb3/3-3/3-3:1.1 Env:map[ACTION:add DEVPATH:/dev>
Aug 03 09:44:08 archlinux kernel: sched_clock: Marking unstable (2559481551, 3758)<-(2564320731, -6528533)
Aug 03 09:44:08 archlinux kernel: TSC found unstable after boot, most likely due to broken BIOS. Use 'tsc=unstable'.
Aug 03 09:44:08 archlinux kernel: clocksource: Switched to clocksource tsc
Aug 03 09:44:08 archlinux kernel: tsc: Marking TSC unstable due to clocksource watchdog
Aug 03 09:44:08 archlinux kernel: clocksource:                       'tsc-early' (not 'tsc') is current clocksource.
Aug 03 09:44:08 archlinux kernel: clocksource:                       'tsc' cs_nsec: 505771997 cs_now: 4fe539f51 cs_last: 4b900afcd mask: ffffffffffffffff
Aug 03 09:44:08 archlinux kernel: clocksource:                       'hpet' wd_nsec: 506629892 wd_now: 1bc26cc wd_last: 14d76ca mask: ffffffff
Aug 03 09:44:08 archlinux kernel: clocksource: timekeeping watchdog on CPU3: Marking clocksource 'tsc' as unstable because the skew is too large:
Aug 03 09:44:08 archlinux kernel: clocksource: tsc: mask: 0xffffffffffffffff max_cycles: 0x2125a853cde, max_idle_ns: 440795219250 ns
Aug 03 09:44:08 archlinux kernel: tsc: Refined TSC clocksource calibration: 2299.580 MHz
Aug 03 09:44:08 archlinux booster: modalias usb:v1050p0402d0543dc00dsc00dp00ic03isc00ip00in00 matched module usbhid
Aug 03 09:44:08 archlinux booster: udev event {Action:bind KObj:/devices/pci0000:00/0000:00:08.1/0000:03:00.3/usb1/1-1 Env:map[ACTION:bind BUSNUM:001 DEVNAME>
Aug 03 09:44:08 archlinux booster: no match found for alias usb:v0BDAp2852d0000dcE0dsc01dp01icE0isc01ip01in00
Aug 03 09:44:08 archlinux booster: udev event {Action:add KObj:/devices/pci0000:00/0000:00:08.1/0000:03:00.3/usb1/1-1/1-1:1.0 Env:map[ACTION:add DEVPATH:/dev>
Aug 03 09:44:08 archlinux booster: udev event {Action:add KObj:/devices/pci0000:00/0000:00:08.1/0000:03:00.4/usb3/3-3/3-3:1.0 Env:map[ACTION:add DEVPATH:/dev>
Aug 03 09:44:08 archlinux booster: udev event {Action:change KObj:/devices/pci0000:00/0000:00:08.1/0000:03:00.4/usb3/3-3 Env:map[ACTION:change BUSNUM:003 DEV>
Aug 03 09:44:08 archlinux booster: udev event {Action:add KObj:/devices/pci0000:00/0000:00:08.1/0000:03:00.3/usb1/1-1 Env:map[ACTION:add BUSNUM:001 DEVNAME:b>
Aug 03 09:44:08 archlinux kernel: usb 1-1: Manufacturer: Yubico
Aug 03 09:44:08 archlinux kernel: usb 1-1: Product: YubiKey FIDO
Aug 03 09:44:08 archlinux kernel: usb 1-1: New USB device strings: Mfr=1, Product=2, SerialNumber=0
Aug 03 09:44:08 archlinux kernel: usb 1-1: New USB device found, idVendor=1050, idProduct=0402, bcdDevice= 5.43
Aug 03 09:44:08 archlinux booster: udev event {Action:add KObj:/devices/pci0000:00/0000:00:08.1/0000:03:00.4/usb3/3-3 Env:map[ACTION:add BUSNUM:003 DEVNAME:b>
Aug 03 09:44:08 archlinux kernel: usb 3-3: SerialNumber: 00e04c000001
Aug 03 09:44:08 archlinux kernel: usb 3-3: Manufacturer: Realtek
Aug 03 09:44:08 archlinux kernel: usb 3-3: Product: Bluetooth Radio
Aug 03 09:44:08 archlinux kernel: usb 3-3: New USB device strings: Mfr=1, Product=2, SerialNumber=3
Aug 03 09:44:08 archlinux kernel: usb 3-3: New USB device found, idVendor=0bda, idProduct=2852, bcdDevice= 0.00
Aug 03 09:44:08 archlinux booster: udev event {Action:add KObj:/module/xhci_pci Env:map[ACTION:add DEVPATH:/module/xhci_pci SEQNUM:1494 SUBSYSTEM:module]}
Aug 03 09:44:08 archlinux booster: udev event {Action:add KObj:/bus/pci/drivers/xhci_hcd Env:map[ACTION:add DEVPATH:/bus/pci/drivers/xhci_hcd SEQNUM:1493 SUB>
Aug 03 09:44:08 archlinux booster: udev event {Action:bind KObj:/devices/pci0000:00/0000:00:08.1/0000:03:00.4 Env:map[ACTION:bind DEVPATH:/devices/pci0000:00>
Aug 03 09:44:08 archlinux booster: udev event {Action:bind KObj:/devices/pci0000:00/0000:00:08.1/0000:03:00.4/usb4 Env:map[ACTION:bind BUSNUM:004 DEVNAME:bus>
Aug 03 09:44:08 archlinux booster: udev event {Action:bind KObj:/devices/pci0000:00/0000:00:08.1/0000:03:00.4/usb4/4-0:1.0 Env:map[ACTION:bind DEVPATH:/devic>
Aug 03 09:44:08 archlinux booster: udev event {Action:add KObj:/devices/pci0000:00/0000:00:08.1/0000:03:00.4/usb4/4-0:1.0 Env:map[ACTION:add DEVPATH:/devices>
Aug 03 09:44:08 archlinux booster: udev event {Action:add KObj:/devices/pci0000:00/0000:00:08.1/0000:03:00.4/usb4 Env:map[ACTION:add BUSNUM:004 DEVNAME:bus/u>
Aug 03 09:44:08 archlinux booster: udev event {Action:bind KObj:/devices/pci0000:00/0000:00:08.1/0000:03:00.4/usb3 Env:map[ACTION:bind BUSNUM:003 DEVNAME:bus>
Aug 03 09:44:08 archlinux booster: udev event {Action:bind KObj:/devices/pci0000:00/0000:00:08.1/0000:03:00.4/usb3/3-0:1.0 Env:map[ACTION:bind DEVPATH:/devic>
Aug 03 09:44:08 archlinux booster: udev event {Action:add KObj:/devices/pci0000:00/0000:00:08.1/0000:03:00.4/usb3/3-0:1.0 Env:map[ACTION:add DEVPATH:/devices>
Aug 03 09:44:08 archlinux booster: udev event {Action:add KObj:/devices/pci0000:00/0000:00:08.1/0000:03:00.4/usb3 Env:map[ACTION:add BUSNUM:003 DEVNAME:bus/u>
Aug 03 09:44:08 archlinux booster: udev event {Action:add KObj:/devices/pci0000:00/0000:00:08.1/0000:03:00.4/wakeup/wakeup46 Env:map[ACTION:add DEVPATH:/devi>
Aug 03 09:44:08 archlinux booster: udev event {Action:bind KObj:/devices/pci0000:00/0000:00:08.1/0000:03:00.3 Env:map[ACTION:bind DEVPATH:/devices/pci0000:00>
Aug 03 09:44:08 archlinux booster: udev event {Action:bind KObj:/devices/pci0000:00/0000:00:08.1/0000:03:00.3/usb2 Env:map[ACTION:bind BUSNUM:002 DEVNAME:bus>
Aug 03 09:44:08 archlinux booster: udev event {Action:bind KObj:/devices/pci0000:00/0000:00:08.1/0000:03:00.3/usb2/2-0:1.0 Env:map[ACTION:bind DEVPATH:/devic>Aug 03 09:44:08 archlinux booster: udev event {Action:add KObj:/module/nvme_core Env:map[ACTION:add DEVPATH:/module/nvme_core SEQNUM:1479 SUBSYSTEM:module]}
Aug 03 09:44:08 archlinux booster: udev event {Action:add KObj:/class/nvme-generic Env:map[ACTION:add DEVPATH:/class/nvme-generic SEQNUM:1478 SUBSYSTEM:class>
Aug 03 09:44:08 archlinux booster: udev event {Action:add KObj:/class/nvme-subsystem Env:map[ACTION:add DEVPATH:/class/nvme-subsystem SEQNUM:1477 SUBSYSTEM:c>
Aug 03 09:44:08 archlinux booster: udev event {Action:add KObj:/class/nvme Env:map[ACTION:add DEVPATH:/class/nvme SEQNUM:1476 SUBSYSTEM:class]}
Aug 03 09:44:08 archlinux booster: udev event {Action:add KObj:/devices/virtual/workqueue/nvme-delete-wq Env:map[ACTION:add DEVPATH:/devices/virtual/workqueu>
Aug 03 09:44:08 archlinux booster: udev event {Action:add KObj:/devices/virtual/workqueue/nvme-reset-wq Env:map[ACTION:add DEVPATH:/devices/virtual/workqueue>
Aug 03 09:44:08 archlinux booster: no match found for alias usb:v1D6Bp0003d0601dc09dsc00dp03ic09isc00ip00in00
Aug 03 09:44:08 archlinux booster: udev event {Action:add KObj:/devices/virtual/workqueue/nvme-wq Env:map[ACTION:add DEVPATH:/devices/virtual/workqueue/nvme->
Aug 03 09:44:08 archlinux booster: udev event {Action:add KObj:/devices/pci0000:00/0000:00:08.1/0000:03:00.3/usb2/2-0:1.0 Env:map[ACTION:add DEVPATH:/devices>
Aug 03 09:44:08 archlinux booster: udev event {Action:add KObj:/devices/pci0000:00/0000:00:08.1/0000:03:00.3/usb2 Env:map[ACTION:add BUSNUM:002 DEVNAME:bus/u>
Aug 03 09:44:08 archlinux booster: udev event {Action:add KObj:/module/crypto_simd Env:map[ACTION:add DEVPATH:/module/crypto_simd SEQNUM:1470 SUBSYSTEM:modul>
Aug 03 09:44:08 archlinux booster: udev event {Action:bind KObj:/devices/pci0000:00/0000:00:08.1/0000:03:00.3/usb1 Env:map[ACTION:bind BUSNUM:001 DEVNAME:bus>
Aug 03 09:44:08 archlinux booster: no match found for alias usb:v1D6Bp0002d0601dc09dsc00dp01ic09isc00ip00in00
Aug 03 09:44:08 archlinux booster: udev event {Action:bind KObj:/devices/pci0000:00/0000:00:08.1/0000:03:00.3/usb1/1-0:1.0 Env:map[ACTION:bind DEVPATH:/devic>
Aug 03 09:44:08 archlinux booster: udev event {Action:add KObj:/devices/pci0000:00/0000:00:08.1/0000:03:00.3/usb1/1-0:1.0 Env:map[ACTION:add DEVPATH:/devices>
Aug 03 09:44:08 archlinux booster: udev event {Action:add KObj:/devices/pci0000:00/0000:00:08.1/0000:03:00.3/usb1 Env:map[ACTION:add BUSNUM:001 DEVNAME:bus/u>
Aug 03 09:44:08 archlinux booster: udev event {Action:add KObj:/devices/pci0000:00/0000:00:08.1/0000:03:00.3/wakeup/wakeup45 Env:map[ACTION:add DEVPATH:/devi>
Aug 03 09:44:08 archlinux booster: udev event {Action:add KObj:/module/ghash_clmulni_intel Env:map[ACTION:add DEVPATH:/module/ghash_clmulni_intel SEQNUM:1464>
Aug 03 09:44:08 archlinux booster: udev event {Action:add KObj:/module/sha512_ssse3 Env:map[ACTION:add DEVPATH:/module/sha512_ssse3 SEQNUM:1463 SUBSYSTEM:mod>
Aug 03 09:44:08 archlinux booster: udev event {Action:add KObj:/module/crc32_pclmul Env:map[ACTION:add DEVPATH:/module/crc32_pclmul SEQNUM:1462 SUBSYSTEM:mod>
Aug 03 09:44:08 archlinux booster: udev event {Action:add KObj:/module/crc32c_intel Env:map[ACTION:add DEVPATH:/module/crc32c_intel SEQNUM:1461 SUBSYSTEM:mod>
Aug 03 09:44:08 archlinux booster: udev event {Action:add KObj:/module/gf128mul Env:map[ACTION:add DEVPATH:/module/gf128mul SEQNUM:1460 SUBSYSTEM:module]}
Aug 03 09:44:08 archlinux booster: udev event {Action:add KObj:/module/amd_sfh Env:map[ACTION:add DEVPATH:/module/amd_sfh SEQNUM:1459 SUBSYSTEM:module]}
Aug 03 09:44:08 archlinux booster: udev event {Action:add KObj:/bus/pci/drivers/pcie_mp2_amd Env:map[ACTION:add DEVPATH:/bus/pci/drivers/pcie_mp2_amd SEQNUM:>
Aug 03 09:44:08 archlinux booster: udev event {Action:remove KObj:/devices/0020:1022:0001.0004 Env:map[ACTION:remove DEVPATH:/devices/0020:1022:0001.0004 HID>
Aug 03 09:44:08 archlinux booster: udev event {Action:unbind KObj:/devices/0020:1022:0001.0004 Env:map[ACTION:unbind DEVPATH:/devices/0020:1022:0001.0004 HID>
Aug 03 09:44:08 archlinux booster: udev event {Action:remove KObj:/devices/0020:1022:0001.0004/hidraw/hidraw3 Env:map[ACTION:remove DEVNAME:hidraw3 DEVPATH:/>
Aug 03 09:44:08 archlinux booster: udev event {Action:remove KObj:/devices/0020:1022:0001.0003 Env:map[ACTION:remove DEVPATH:/devices/0020:1022:0001.0003 HID>
Aug 03 09:44:08 archlinux booster: udev event {Action:unbind KObj:/devices/0020:1022:0001.0003 Env:map[ACTION:unbind DEVPATH:/devices/0020:1022:0001.0003 HID>
Aug 03 09:44:08 archlinux booster: udev event {Action:remove KObj:/devices/0020:1022:0001.0003/hidraw/hidraw2 Env:map[ACTION:remove DEVNAME:hidraw2 DEVPATH:/>
Aug 03 09:44:08 archlinux booster: udev event {Action:remove KObj:/devices/0020:1022:0001.0002 Env:map[ACTION:remove DEVPATH:/devices/0020:1022:0001.0002 HID>
Aug 03 09:44:08 archlinux booster: udev event {Action:unbind KObj:/devices/0020:1022:0001.0002 Env:map[ACTION:unbind DEVPATH:/devices/0020:1022:0001.0002 HID>
Aug 03 09:44:08 archlinux booster: udev event {Action:remove KObj:/devices/0020:1022:0001.0002/hidraw/hidraw1 Env:map[ACTION:remove DEVNAME:hidraw1 DEVPATH:/>
Aug 03 09:44:08 archlinux kernel: usb 3-3: new full-speed USB device number 2 using xhci_hcd
Aug 03 09:44:08 archlinux kernel: usb 1-1: new full-speed USB device number 2 using xhci_hcd
Aug 03 09:44:08 archlinux booster: udev event {Action:bind KObj:/devices/0020:1022:0001.0004 Env:map[ACTION:bind DEVPATH:/devices/0020:1022:0001.0004 DRIVER:>
Aug 03 09:44:08 archlinux booster: udev event {Action:add KObj:/devices/0020:1022:0001.0004/hidraw/hidraw3 Env:map[ACTION:add DEVNAME:hidraw3 DEVPATH:/device>
Aug 03 09:44:08 archlinux booster: udev event {Action:add KObj:/devices/0020:1022:0001.0004 Env:map[ACTION:add DEVPATH:/devices/0020:1022:0001.0004 HID_ID:00>
Aug 03 09:44:08 archlinux booster: udev event {Action:bind KObj:/devices/0020:1022:0001.0003 Env:map[ACTION:bind DEVPATH:/devices/0020:1022:0001.0003 DRIVER:>
Aug 03 09:44:08 archlinux booster: loading module hid_sensor_hub
Aug 03 09:44:08 archlinux booster: udev event {Action:add KObj:/devices/0020:1022:0001.0003/hidraw/hidraw2 Env:map[ACTION:add DEVNAME:hidraw2 DEVPATH:/device>
Aug 03 09:44:08 archlinux booster: udev event {Action:add KObj:/devices/0020:1022:0001.0003 Env:map[ACTION:add DEVPATH:/devices/0020:1022:0001.0003 HID_ID:00>
Aug 03 09:44:08 archlinux booster: udev event {Action:bind KObj:/devices/0020:1022:0001.0002 Env:map[ACTION:bind DEVPATH:/devices/0020:1022:0001.0002 DRIVER:>
Aug 03 09:44:08 archlinux booster: modalias hid:b0020g0003v00001022p00000001 matched module hid_sensor_hub
Aug 03 09:44:08 archlinux booster: udev event {Action:add KObj:/devices/0020:1022:0001.0002/hidraw/hidraw1 Env:map[ACTION:add DEVNAME:hidraw1 DEVPATH:/device>
Aug 03 09:44:08 archlinux booster: udev event {Action:add KObj:/devices/0020:1022:0001.0002 Env:map[ACTION:add DEVPATH:/devices/0020:1022:0001.0002 HID_ID:00>
Aug 03 09:44:08 archlinux booster: loading module vivaldi_fmap
Aug 03 09:44:08 archlinux booster: loading module libps2
Aug 03 09:44:08 archlinux booster: loading module serio_raw
Aug 03 09:44:08 archlinux booster: udev event {Action:add KObj:/module/cryptd Env:map[ACTION:add DEVPATH:/module/cryptd SEQNUM:1439 SUBSYSTEM:module]}
Aug 03 09:44:08 archlinux booster: modalias serio:ty06pr00id00ex00 matched module serio_raw
Aug 03 09:44:08 archlinux booster: udev event {Action:add KObj:/module/crct10dif_pclmul Env:map[ACTION:add DEVPATH:/module/crct10dif_pclmul SEQNUM:1438 SUBSY>
Aug 03 09:44:08 archlinux booster: udev event {Action:add KObj:/module/i8042 Env:map[ACTION:add DEVPATH:/module/i8042 SEQNUM:1437 SUBSYSTEM:module]}
Aug 03 09:44:08 archlinux booster: modalias serio:ty06pr00id00ex00 matched module atkbd
Aug 03 09:44:08 archlinux booster: udev event {Action:bind KObj:/devices/platform/i8042 Env:map[ACTION:bind DEVPATH:/devices/platform/i8042 DRIVER:i8042 MODA>
Aug 03 09:44:08 archlinux booster: udev event {Action:add KObj:/devices/platform/i8042/serio0 Env:map[ACTION:add DEVPATH:/devices/platform/i8042/serio0 MODAL>
Aug 03 09:44:08 archlinux booster: no match found for alias platform:i8042
Aug 03 09:44:08 archlinux booster: udev event {Action:add KObj:/devices/platform/i8042/serio0/wakeup/wakeup43 Env:map[ACTION:add DEVPATH:/devices/platform/i8>
Aug 03 09:44:08 archlinux booster: udev event {Action:add KObj:/devices/platform/i8042 Env:map[ACTION:add DEVPATH:/devices/platform/i8042 MODALIAS:platform:i>
Aug 03 09:44:08 archlinux booster: udev event {Action:add KObj:/bus/platform/drivers/i8042 Env:map[ACTION:add DEVPATH:/bus/platform/drivers/i8042 SEQNUM:1432>
Aug 03 09:44:08 archlinux booster: udev event {Action:add KObj:/bus/pnp/drivers/i8042 aux Env:map[ACTION:add DEVPATH:/bus/pnp/drivers/i8042 aux SEQNUM:1431 S>
Aug 03 09:44:08 archlinux booster: udev event {Action:add KObj:/bus/pnp/drivers/i8042 kbd Env:map[ACTION:add DEVPATH:/bus/pnp/drivers/i8042 kbd SEQNUM:1430 S>
Aug 03 09:44:08 archlinux booster: udev event {Action:bind KObj:/devices/pnp0/00:02 Env:map[ACTION:bind DEVPATH:/devices/pnp0/00:02 DRIVER:i8042 kbd SEQNUM:1>
Aug 03 09:44:08 archlinux booster: udev event {Action:add KObj:/module/xhci_pci_renesas Env:map[ACTION:add DEVPATH:/module/xhci_pci_renesas SEQNUM:1428 SUBSY>
Aug 03 09:44:08 archlinux booster: udev event {Action:add KObj:/module/nvme_common Env:map[ACTION:add DEVPATH:/module/nvme_common SEQNUM:1427 SUBSYSTEM:modul>
Aug 03 09:44:08 archlinux booster: udev event {Action:bind KObj:/devices/platform/AMDI0010:03/i2c-0/i2c-ELAN074E:00 Env:map[ACTION:bind DEVPATH:/devices/plat>
Aug 03 09:44:08 archlinux booster: loading module hid_multitouch
Aug 03 09:44:08 archlinux booster: udev event {Action:bind KObj:/devices/platform/AMDI0010:03/i2c-0/i2c-ELAN074E:00/0018:04F3:31B5.0001 Env:map[ACTION:bind D>
Aug 03 09:44:08 archlinux booster: udev event {Action:add KObj:/devices/platform/AMDI0010:03/i2c-0/i2c-ELAN074E:00/0018:04F3:31B5.0001/hidraw/hidraw0 Env:map>
Aug 03 09:44:08 archlinux booster: no match found for alias input:b0018v04F3p31B5e0100_e0,1,3,4,k100,14A,ra0,1,28,m4,lsfw
Aug 03 09:44:08 archlinux booster: udev event {Action:add KObj:/devices/platform/AMDI0010:03/i2c-0/i2c-ELAN074E:00/0018:04F3:31B5.0001/input/input5/event4 En>
Aug 03 09:44:08 archlinux booster: udev event {Action:add KObj:/devices/platform/AMDI0010:03/i2c-0/i2c-ELAN074E:00/0018:04F3:31B5.0001/input/input5 Env:map[A>
Aug 03 09:44:08 archlinux booster: no match found for alias input:b0018v04F3p31B5e0100_e0,1,2,4,k110,111,r0,1,am4,lsfw
Aug 03 09:44:08 archlinux booster: udev event {Action:add KObj:/devices/platform/AMDI0010:03/i2c-0/i2c-ELAN074E:00/0018:04F3:31B5.0001/input/input3/event3 En>
Aug 03 09:44:08 archlinux booster: modalias hid:b0018g0004v000004F3p000031B5 matched module hid_multitouch
Aug 03 09:44:08 archlinux booster: udev event {Action:add KObj:/devices/platform/AMDI0010:03/i2c-0/i2c-ELAN074E:00/0018:04F3:31B5.0001/input/input3 Env:map[A>
Aug 03 09:44:08 archlinux booster: udev event {Action:add KObj:/devices/platform/AMDI0010:03/i2c-0/i2c-ELAN074E:00/0018:04F3:31B5.0001 Env:map[ACTION:add DEV>
Aug 03 09:44:08 archlinux booster: udev event {Action:add KObj:/module/serio Env:map[ACTION:add DEVPATH:/module/serio SEQNUM:1418 SUBSYSTEM:module]}
Aug 03 09:44:08 archlinux booster: udev event {Action:add KObj:/bus/serio Env:map[ACTION:add DEVPATH:/bus/serio SEQNUM:1417 SUBSYSTEM:bus]}
Aug 03 09:44:08 archlinux booster: udev event {Action:remove KObj:/devices/platform/AMDI0010:03/i2c-0/i2c-ELAN074E:00/wakeup/wakeup43 Env:map[ACTION:remove D>
Aug 03 09:44:08 archlinux booster: udev event {Action:add KObj:/devices/LNXSYSTM:00/LNXSYBUS:00/AMDI0010:03/ELAN074E:00/wakeup/wakeup44 Env:map[ACTION:add DE>
Aug 03 09:44:08 archlinux booster: udev event {Action:add KObj:/devices/platform/AMDI0010:03/i2c-0/i2c-ELAN074E:00/wakeup/wakeup43 Env:map[ACTION:add DEVPATH>
Aug 03 09:44:08 archlinux booster: udev event {Action:change KObj:/devices/platform/AMDI0010:03/i2c-0/i2c-ELAN074E:00 Env:map[ACTION:change DEVPATH:/devices/>
Aug 03 09:44:08 archlinux booster: udev event {Action:add KObj:/module/i2c_hid_acpi Env:map[ACTION:add DEVPATH:/module/i2c_hid_acpi SEQNUM:1412 SUBSYSTEM:mod>
Aug 03 09:44:08 archlinux booster: udev event {Action:add KObj:/bus/i2c/drivers/i2c_hid_acpi Env:map[ACTION:add DEVPATH:/bus/i2c/drivers/i2c_hid_acpi SEQNUM:>
Aug 03 09:44:08 archlinux booster: udev event {Action:add KObj:/module/drm_display_helper Env:map[ACTION:add DEVPATH:/module/drm_display_helper SEQNUM:1410 S>
Aug 03 09:44:08 archlinux booster: udev event {Action:add KObj:/class/drm_dp_aux_dev Env:map[ACTION:add DEVPATH:/class/drm_dp_aux_dev SEQNUM:1409 SUBSYSTEM:c>
Aug 03 09:44:08 archlinux booster: udev event {Action:add KObj:/module/drm_ttm_helper Env:map[ACTION:add DEVPATH:/module/drm_ttm_helper SEQNUM:1408 SUBSYSTEM>
Aug 03 09:44:08 archlinux booster: udev event {Action:add KObj:/module/video Env:map[ACTION:add DEVPATH:/module/video SEQNUM:1407 SUBSYSTEM:module]}
Aug 03 09:44:08 archlinux booster: udev event {Action:add KObj:/bus/acpi/drivers/video Env:map[ACTION:add DEVPATH:/bus/acpi/drivers/video SEQNUM:1406 SUBSYST>
Aug 03 09:44:08 archlinux booster: udev event {Action:bind KObj:/devices/LNXSYSTM:00/LNXSYBUS:00/PNP0A08:00/device:0a/LNXVIDEO:00 Env:map[ACTION:bind DEVPATH>
Aug 03 09:44:08 archlinux booster: no match found for alias input:b0019v0000p0006e0000_e0,1,kE0,E1,E3,F1,F2,F3,F4,F5,ramlsfw
Aug 03 09:44:08 archlinux booster: udev event {Action:add KObj:/devices/LNXSYSTM:00/LNXSYBUS:00/PNP0A08:00/device:0a/LNXVIDEO:00/input/input2/event2 Env:map[>
Aug 03 09:44:08 archlinux booster: udev event {Action:add KObj:/devices/LNXSYSTM:00/LNXSYBUS:00/PNP0A08:00/device:0a/LNXVIDEO:00/input/input2 Env:map[ACTION:>
Aug 03 09:44:08 archlinux booster: udev event {Action:add KObj:/module/cec Env:map[ACTION:add DEVPATH:/module/cec SEQNUM:1402 SUBSYSTEM:module]}
Aug 03 09:44:08 archlinux booster: udev event {Action:add KObj:/bus/cec Env:map[ACTION:add DEVPATH:/bus/cec SEQNUM:1401 SUBSYSTEM:bus]}
Aug 03 09:44:08 archlinux booster: udev event {Action:add KObj:/module/i2c_hid Env:map[ACTION:add DEVPATH:/module/i2c_hid SEQNUM:1400 SUBSYSTEM:module]}
Aug 03 09:44:08 archlinux booster: udev event {Action:add KObj:/module/ttm Env:map[ACTION:add DEVPATH:/module/ttm SEQNUM:1399 SUBSYSTEM:module]}
Aug 03 09:44:08 archlinux booster: udev event {Action:add KObj:/module/wmi Env:map[ACTION:add DEVPATH:/module/wmi SEQNUM:1398 SUBSYSTEM:module]}
Aug 03 09:44:08 archlinux booster: udev event {Action:add KObj:/bus/platform/drivers/acpi-wmi Env:map[ACTION:add DEVPATH:/bus/platform/drivers/acpi-wmi SEQNU>
Aug 03 09:44:08 archlinux booster: udev event {Action:bind KObj:/devices/platform/PNP0C14:04 Env:map[ACTION:bind DEVPATH:/devices/platform/PNP0C14:04 DRIVER:
Aug 03 09:44:08 archlinux booster: no match found for alias wmi:ABBC0F6A_8EA1_11D1_00A0_C90629100000
Aug 03 09:44:08 archlinux booster: udev event {Action:add KObj:/devices/platform/PNP0C14:04/wmi_bus/wmi_bus-PNP0C14:04/05901221-D566-11D1-B2F0-00A0C9062910-4>
Aug 03 09:44:08 archlinux booster: udev event {Action:add KObj:/devices/platform/PNP0C14:04/wmi_bus/wmi_bus-PNP0C14:04/ABBC0F6A-8EA1-11D1-00A0-C90629100000 E>
Aug 03 09:44:08 archlinux booster: udev event {Action:add KObj:/module/drm_buddy Env:map[ACTION:add DEVPATH:/module/drm_buddy SEQNUM:1393 SUBSYSTEM:module]}
Aug 03 09:44:08 archlinux booster: udev event {Action:add KObj:/devices/platform/PNP0C14:04/wmi_bus/wmi_bus-PNP0C14:04 Env:map[ACTION:add DEVPATH:/devices/pl>
Aug 03 09:44:08 archlinux booster: udev event {Action:add KObj:/devices/LNXSYSTM:00/PNP0C14:04/wakeup/wakeup42 Env:map[ACTION:add DEVPATH:/devices/LNXSYSTM:0>
Aug 03 09:44:08 archlinux booster: udev event {Action:bind KObj:/devices/platform/PNP0C14:03 Env:map[ACTION:bind DEVPATH:/devices/platform/PNP0C14:03 DRIVER:>
Aug 03 09:44:08 archlinux booster: no match found for alias wmi:40D1BF71_A82D_4E59_A168_3985E03B2E87
Aug 03 09:44:08 archlinux booster: udev event {Action:add KObj:/devices/platform/PNP0C14:03/wmi_bus/wmi_bus-PNP0C14:03/05901221-D566-11D1-B2F0-00A0C9062910-3>
Aug 03 09:44:08 archlinux booster: no match found for alias wmi:431F16ED_0C2B_444C_B267_27DEB140CF9C
Aug 03 09:44:08 archlinux booster: udev event {Action:add KObj:/devices/platform/PNP0C14:03/wmi_bus/wmi_bus-PNP0C14:03/40D1BF71-A82D-4E59-A168-3985E03B2E87 E>
Aug 03 09:44:08 archlinux booster: no match found for alias wmi:67C3371D_95A3_4C37_BB61_DD47B491DAAB
Aug 03 09:44:08 archlinux booster: udev event {Action:add KObj:/devices/platform/PNP0C14:03/wmi_bus/wmi_bus-PNP0C14:03/431F16ED-0C2B-444C-B267-27DEB140CF9C E>
Aug 03 09:44:08 archlinux booster: no match found for alias wmi:D9F41781_F633_4400_9355_601770BEC510
Aug 03 09:44:08 archlinux booster: udev event {Action:add KObj:/devices/platform/PNP0C14:03/wmi_bus/wmi_bus-PNP0C14:03/67C3371D-95A3-4C37-BB61-DD47B491DAAB E>
Aug 03 09:44:08 archlinux booster: udev event {Action:add KObj:/devices/platform/PNP0C14:03/wmi_bus/wmi_bus-PNP0C14:03/D9F41781-F633-4400-9355-601770BEC510 E>
Aug 03 09:44:08 archlinux booster: udev event {Action:add KObj:/devices/platform/PNP0C14:03/wmi_bus/wmi_bus-PNP0C14:03 Env:map[ACTION:add DEVPATH:/devices/pl>
Aug 03 09:44:08 archlinux booster: udev event {Action:add KObj:/devices/LNXSYSTM:00/LNXSYBUS:00/PNP0C14:03/wakeup/wakeup41 Env:map[ACTION:add DEVPATH:/device>
Aug 03 09:44:08 archlinux booster: udev event {Action:bind KObj:/devices/platform/PNP0C14:02 Env:map[ACTION:bind DEVPATH:/devices/platform/PNP0C14:02 DRIVER:>
Aug 03 09:44:08 archlinux booster: no match found for alias wmi:1F13AB7F_6220_4210_8F8E_8BB5E71EE969
Aug 03 09:44:08 archlinux booster: udev event {Action:add KObj:/devices/platform/PNP0C14:02/wmi_bus/wmi_bus-PNP0C14:02/05901221-D566-11D1-B2F0-00A0C9062910-2>
Aug 03 09:44:08 archlinux booster: loading module nvme
Aug 03 09:44:08 archlinux booster: loading module aesni_intel
Aug 03 09:44:08 archlinux kernel: hub 4-0:1.0: 2 ports detected
Aug 03 09:44:08 archlinux kernel: hub 4-0:1.0: USB hub found
Aug 03 09:44:08 archlinux kernel: usb usb4: SerialNumber: 0000:03:00.4
Aug 03 09:44:08 archlinux kernel: usb usb4: Manufacturer: Linux 6.1.39-3-lts xhci-hcd
Aug 03 09:44:08 archlinux kernel: usb usb4: Product: xHCI Host Controller
Aug 03 09:44:08 archlinux kernel: usb usb4: New USB device strings: Mfr=3, Product=2, SerialNumber=1
Aug 03 09:44:08 archlinux kernel: usb usb4: New USB device found, idVendor=1d6b, idProduct=0003, bcdDevice= 6.01
Aug 03 09:44:08 archlinux kernel: usb usb4: We don't know the algorithms for LPM for this host, disabling LPM.
Aug 03 09:44:08 archlinux kernel: hub 3-0:1.0: 4 ports detected
Aug 03 09:44:08 archlinux kernel: hub 3-0:1.0: USB hub found
Aug 03 09:44:08 archlinux kernel: usb usb3: SerialNumber: 0000:03:00.4
Aug 03 09:44:08 archlinux kernel: usb usb3: Manufacturer: Linux 6.1.39-3-lts xhci-hcd
Aug 03 09:44:08 archlinux kernel: usb usb3: Product: xHCI Host Controller
Aug 03 09:44:08 archlinux kernel: usb usb3: New USB device strings: Mfr=3, Product=2, SerialNumber=1
Aug 03 09:44:08 archlinux kernel: usb usb3: New USB device found, idVendor=1d6b, idProduct=0002, bcdDevice= 6.01
Aug 03 09:44:08 archlinux kernel: xhci_hcd 0000:03:00.4: Host supports USB 3.1 Enhanced SuperSpeed
Aug 03 09:44:08 archlinux kernel: xhci_hcd 0000:03:00.4: new USB bus registered, assigned bus number 4
Aug 03 09:44:08 archlinux kernel: xhci_hcd 0000:03:00.4: xHCI Host Controller
Aug 03 09:44:08 archlinux booster: udev event {Action:add KObj:/devices/platform/PNP0C14:02/wmi_bus/wmi_bus-PNP0C14:02/1F13AB7F-6220-4210-8F8E-8BB5E71EE969 E>
Aug 03 09:44:08 archlinux kernel: xhci_hcd 0000:03:00.4: hcc params 0x0268ffe5 hci version 0x110 quirks 0x0000020000000410
Aug 03 09:44:08 archlinux kernel: xhci_hcd 0000:03:00.4: new USB bus registered, assigned bus number 3
Aug 03 09:44:08 archlinux kernel: xhci_hcd 0000:03:00.4: xHCI Host Controller
Aug 03 09:44:08 archlinux kernel: hub 2-0:1.0: 2 ports detected
Aug 03 09:44:08 archlinux kernel: hub 2-0:1.0: USB hub found
Aug 03 09:44:08 archlinux kernel: usb usb2: SerialNumber: 0000:03:00.3
Aug 03 09:44:08 archlinux kernel: usb usb2: Manufacturer: Linux 6.1.39-3-lts xhci-hcd
Aug 03 09:44:08 archlinux kernel: usb usb2: Product: xHCI Host Controller
Aug 03 09:44:08 archlinux kernel: usb usb2: New USB device strings: Mfr=3, Product=2, SerialNumber=1
Aug 03 09:44:08 archlinux kernel: usb usb2: New USB device found, idVendor=1d6b, idProduct=0003, bcdDevice= 6.01
Aug 03 09:44:08 archlinux kernel: usb usb2: We don't know the algorithms for LPM for this host, disabling LPM.
Aug 03 09:44:08 archlinux kernel: hub 1-0:1.0: 4 ports detected
Aug 03 09:44:08 archlinux kernel: hub 1-0:1.0: USB hub found
Aug 03 09:44:08 archlinux kernel: usb usb1: SerialNumber: 0000:03:00.3
Aug 03 09:44:08 archlinux kernel: usb usb1: Manufacturer: Linux 6.1.39-3-lts xhci-hcd
Aug 03 09:44:08 archlinux kernel: usb usb1: Product: xHCI Host Controller
Aug 03 09:44:08 archlinux kernel: usb usb1: New USB device strings: Mfr=3, Product=2, SerialNumber=1
Aug 03 09:44:08 archlinux kernel: usb usb1: New USB device found, idVendor=1d6b, idProduct=0002, bcdDevice= 6.01
Aug 03 09:44:08 archlinux kernel: xhci_hcd 0000:03:00.3: Host supports USB 3.1 Enhanced SuperSpeed
Aug 03 09:44:08 archlinux kernel: xhci_hcd 0000:03:00.3: new USB bus registered, assigned bus number 2
Aug 03 09:44:08 archlinux kernel: xhci_hcd 0000:03:00.3: xHCI Host Controller
Aug 03 09:44:08 archlinux booster: udev event {Action:add KObj:/devices/platform/PNP0C14:02/wmi_bus/wmi_bus-PNP0C14:02 Env:map[ACTION:add DEVPATH:/devices/pl>
Aug 03 09:44:08 archlinux kernel: xhci_hcd 0000:03:00.3: hcc params 0x0268ffe5 hci version 0x110 quirks 0x0000020000000410
Aug 03 09:44:08 archlinux kernel: xhci_hcd 0000:03:00.3: new USB bus registered, assigned bus number 1
Aug 03 09:44:08 archlinux kernel: xhci_hcd 0000:03:00.3: xHCI Host Controller
Aug 03 09:44:08 archlinux booster: loading module polyval_generic
Aug 03 09:44:08 archlinux kernel: pcie_mp2_amd: probe of 0000:03:00.7 failed with error -95
Aug 03 09:44:08 archlinux kernel: pcie_mp2_amd 0000:03:00.7: Failed to discover, sensors not enabled is 1
Aug 03 09:44:08 archlinux booster: udev event {Action:add KObj:/devices/LNXSYSTM:00/LNXSYBUS:00/PNP0C14:02/wakeup/wakeup40 Env:map[ACTION:add DEVPATH:/device>
Aug 03 09:44:08 archlinux booster: udev event {Action:bind KObj:/devices/platform/PNP0C14:01 Env:map[ACTION:bind DEVPATH:/devices/platform/PNP0C14:01 DRIVER:>
Aug 03 09:44:08 archlinux booster: no match found for alias wmi:A6FEA33E_DABF_46F5_BFC8_460D961BEC9F
Aug 03 09:44:08 archlinux booster: udev event {Action:add KObj:/devices/platform/PNP0C14:01/wmi_bus/wmi_bus-PNP0C14:01/05901221-D566-11D1-B2F0-00A0C9062910-1>
Aug 03 09:44:08 archlinux kernel: hid-generic 0020:1022:0001.0004: hidraw3: SENSOR HUB HID v0.00 Device [hid-amdsfh 1022:0001] on pcie_mp2_amd
Aug 03 09:44:08 archlinux booster: no match found for alias wmi:2BC49DEF_7B15_4F05_8BB7_EE37B9547C0B
Aug 03 09:44:08 archlinux booster: udev event {Action:add KObj:/devices/platform/PNP0C14:01/wmi_bus/wmi_bus-PNP0C14:01/A6FEA33E-DABF-46F5-BFC8-460D961BEC9F E>
Aug 03 09:44:08 archlinux kernel: hid-generic 0020:1022:0001.0003: hidraw2: SENSOR HUB HID v0.00 Device [hid-amdsfh 1022:0001] on pcie_mp2_amd
Aug 03 09:44:08 archlinux booster: udev event {Action:add KObj:/devices/platform/PNP0C14:01/wmi_bus/wmi_bus-PNP0C14:01/2BC49DEF-7B15-4F05-8BB7-EE37B9547C0B E>
Aug 03 09:44:08 archlinux kernel: hid-generic 0020:1022:0001.0002: hidraw1: SENSOR HUB HID v0.00 Device [hid-amdsfh 1022:0001] on pcie_mp2_amd
Aug 03 09:44:08 archlinux booster: udev event {Action:add KObj:/devices/platform/PNP0C14:01/wmi_bus/wmi_bus-PNP0C14:01 Env:map[ACTION:add DEVPATH:/devices/pl>
Aug 03 09:44:08 archlinux booster: udev event {Action:add KObj:/devices/LNXSYSTM:00/LNXSYBUS:00/PNP0C14:01/wakeup/wakeup39 Env:map[ACTION:add DEVPATH:/device>
Aug 03 09:44:08 archlinux booster: no match found for alias wmi:DF4E63B6_3BBC_4858_9737_C74F82F821F3
Aug 03 09:44:08 archlinux booster: udev event {Action:bind KObj:/devices/platform/PNP0C14:00 Env:map[ACTION:bind DEVPATH:/devices/platform/PNP0C14:00 DRIVER:>
Aug 03 09:44:08 archlinux booster: no match found for alias wmi:7391A661_223A_47DB_A77A_7BE84C60822D
Aug 03 09:44:08 archlinux booster: udev event {Action:add KObj:/devices/platform/PNP0C14:00/wmi_bus/wmi_bus-PNP0C14:00/DF4E63B6-3BBC-4858-9737-C74F82F821F3 E>
Aug 03 09:44:08 archlinux booster: loading module crypto_simd
Aug 03 09:44:08 archlinux booster: loading module ghash_clmulni_intel
Aug 03 09:44:08 archlinux kernel: cryptd: max_cpu_qlen set to 1000
Aug 03 09:44:08 archlinux booster: no match found for alias wmi:8F1F6435_9F42_42C8_BADC_0E9424F20C9A
Aug 03 09:44:08 archlinux kernel: serio: i8042 KBD port at 0x60,0x64 irq 1
Aug 03 09:44:08 archlinux booster: udev event {Action:add KObj:/devices/platform/PNP0C14:00/wmi_bus/wmi_bus-PNP0C14:00/7391A661-223A-47DB-A77A-7BE84C60822D E>
Aug 03 09:44:08 archlinux kernel: pcie_mp2_amd 0000:03:00.7: enabling device (0000 -> 0002)
Aug 03 09:44:08 archlinux kernel: i8042: PNP: PS/2 appears to have AUX port disabled, if this is incorrect please boot with i8042.nopnp
Aug 03 09:44:08 archlinux kernel: i8042: PNP: PS/2 Controller [PNP0303:KBD0] at 0x60,0x64 irq 1
Aug 03 09:44:08 archlinux booster: loading module xhci_pci
Aug 03 09:44:08 archlinux booster: loading module nvme_core
Aug 03 09:44:08 archlinux booster: loading module crc32_pclmul
Aug 03 09:44:08 archlinux booster: loading module crc32c_intel
Aug 03 09:44:08 archlinux booster: loading module gf128mul
Aug 03 09:44:08 archlinux booster: loading module sha512_ssse3
Aug 03 09:44:08 archlinux booster: loading module cryptd
Aug 03 09:44:08 archlinux booster: loading module crct10dif_pclmul
Aug 03 09:44:08 archlinux booster: modalias cpu:type:x86,ven0002fam0019mod0050:feature:,0000,0001,0002,0003,0004,0005,0006,0007,0008,0009,000B,000C,000D,000E>
Aug 03 09:44:08 archlinux booster: modalias cpu:type:x86,ven0002fam0019mod0050:feature:,0000,0001,0002,0003,0004,0005,0006,0007,0008,0009,000B,000C,000D,000E>
Aug 03 09:44:08 archlinux booster: modalias cpu:type:x86,ven0002fam0019mod0050:feature:,0000,0001,0002,0003,0004,0005,0006,0007,0008,0009,000B,000C,000D,000E>
Aug 03 09:44:08 archlinux booster: modalias cpu:type:x86,ven0002fam0019mod0050:feature:,0000,0001,0002,0003,0004,0005,0006,0007,0008,0009,000B,000C,000D,000E>
Aug 03 09:44:08 archlinux booster: loading module i8042
Aug 03 09:44:08 archlinux kernel: hid-generic 0018:04F3:31B5.0001: input,hidraw0: I2C HID v1.00 Mouse [ELAN074E:00 04F3:31B5] on i2c-ELAN074E:00
Aug 03 09:44:08 archlinux kernel: input: ELAN074E:00 04F3:31B5 Touchpad as /devices/platform/AMDI0010:03/i2c-0/i2c-ELAN074E:00/0018:04F3:31B5.0001/input/inpu>
Aug 03 09:44:08 archlinux kernel: input: ELAN074E:00 04F3:31B5 Mouse as /devices/platform/AMDI0010:03/i2c-0/i2c-ELAN074E:00/0018:04F3:31B5.0001/input/input3
Aug 03 09:44:08 archlinux booster: modalias cpu:type:x86,ven0002fam0019mod0050:feature:,0000,0001,0002,0003,0004,0005,0006,0007,0008,0009,000B,000C,000D,000E>
Aug 03 09:44:08 archlinux booster: modalias cpu:type:x86,ven0002fam0019mod0050:feature:,0000,0001,0002,0003,0004,0005,0006,0007,0008,0009,000B,000C,000D,000E>
Aug 03 09:44:08 archlinux booster: modalias cpu:type:x86,ven0002fam0019mod0050:feature:,0000,0001,0002,0003,0004,0005,0006,0007,0008,0009,000B,000C,000D,000E>
Aug 03 09:44:08 archlinux booster: modalias cpu:type:x86,ven0002fam0019mod0050:feature:,0000,0001,0002,0003,0004,0005,0006,0007,0008,0009,000B,000C,000D,000E>
Aug 03 09:44:08 archlinux booster: no match found for alias wmi:8F1F6436_9F42_42C8_BADC_0E9424F20C9A
Aug 03 09:44:08 archlinux booster: udev event {Action:add KObj:/devices/platform/PNP0C14:00/wmi_bus/wmi_bus-PNP0C14:00/8F1F6435-9F42-42C8-BADC-0E9424F20C9A E>
Aug 03 09:44:08 archlinux booster: no match found for alias dmi:bvnAMI:bvrF.04:bd07/28/2021:br15.4:efr79.29:svnHP:pnHPPavilionAeroLaptop13-be0xxx:pvr:rvnHP:r>
Aug 03 09:44:08 archlinux booster: loading module amd_sfh
Aug 03 09:44:08 archlinux booster: loading module xhci_pci_renesas
Aug 03 09:44:08 archlinux booster: loading module nvme_common
Aug 03 09:44:08 archlinux booster: modalias pci:v00001C5Cd0000174Asv00001C5Csd0000174Abc01sc08i02 matched module nvme
Aug 03 09:44:08 archlinux booster: modalias pci:v00001002d00001638sv0000103Csd00008919bc03sc00i00 matched module amdgpu
Aug 03 09:44:08 archlinux booster: modalias cpu:type:x86,ven0002fam0019mod0050:feature:,0000,0001,0002,0003,0004,0005,0006,0007,0008,0009,000B,000C,000D,000E>
Aug 03 09:44:08 archlinux booster: loading module serio

</pre>
</details>

Reply:
Sorry for the misunderstanding, but I just corrected the above .yaml files. Neither had "usbhid" and "hid_sensor_hub" modules force loaded. Still, it's observed the modules are loaded and its associated udev events occur before the key is asked for a universal images.